{"failedTests":[{"testName":"parallel/test-cli-permission-deny-fs","timestamp":1701217871799,"status":"REGRESSION","jobName":"node-test-binary-windows-js-suites","config":"node-test-binary-windows-js-suites » 3,win2022-COMPILED_BY-vs2022 #24735","buildUrl":"https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=3,nodes=win2022-COMPILED_BY-vs2022/24735/","buildNumber":24735,"builtOn":"test-azure_msft-win2022_vs2019-x64-3","nodeVersion":"22.0.0","commitHash":"b2649a3bb95e6a322396f5c6e4a70a9e4a9c110e","callStack":[{"upstreamBuild":59529,"upstreamProject":"node-test-commit-windows-fanned","upstreamUrl":"job/node-test-commit-windows-fanned/"},{"upstreamBuild":66977,"upstreamProject":"node-test-commit","upstreamUrl":"job/node-test-commit/"},{"upstreamBuild":55992,"upstreamProject":"node-test-pull-request","upstreamUrl":"job/node-test-pull-request/"},{"prId":"50758"}],"tap":{"ok":false,"name":"parallel/test-cli-permission-deny-fs","id":130,"buffered":false,"tapError":null,"skip":false,"todo":false,"previous":null,"plan":null,"diag":{"severity":"fail","exitcode":1,"stack":"node:assert:1035\n    throw err;\n    ^\n\nAssertionError [ERR_ASSERTION]: The input did not match the regular expression /resource: '.*?[\\\\/](?:etc|passwd)'/. Input:\n\n'(node:1824) ExperimentalWarning: Permission is an experimental feature\\n' +\n  '(Use `node --trace-warnings ...` to show where the warning was created)\\n'\n\n    at Object.<anonymous> (C:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\parallel\\test-cli-permission-deny-fs.js:150:10)\n    at Module._compile (node:internal/modules/cjs/loader:1375:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1434:10)\n    at Module.load (node:internal/modules/cjs/loader:1206:32)\n    at Module._load (node:internal/modules/cjs/loader:1022:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:142:12)\n    at node:internal/main/run_main_module:28:49 {\n  generatedMessage: true,\n  code: 'ERR_ASSERTION',\n  actual: '(node:1824) ExperimentalWarning: Permission is an experimental feature\\n' +\n    '(Use `node --trace-warnings ...` to show where the warning was created)\\n',\n  expected: /resource: '.*?[\\\\/](?:etc|passwd)'/,\n  operator: 'match'\n}\n\nNode.js v22.0.0-pre"},"time":2795.016,"fullname":"parallel/test-cli-permission-deny-fs","closingTestPoint":false}},{"testName":"parallel/test-permission-fs-wildcard","timestamp":1701217871799,"status":"REGRESSION","jobName":"node-test-binary-windows-js-suites","config":"node-test-binary-windows-js-suites » 3,win2022-COMPILED_BY-vs2022 #24735","buildUrl":"https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=3,nodes=win2022-COMPILED_BY-vs2022/24735/","buildNumber":24735,"builtOn":"test-azure_msft-win2022_vs2019-x64-3","nodeVersion":"22.0.0","commitHash":"b2649a3bb95e6a322396f5c6e4a70a9e4a9c110e","callStack":[{"upstreamBuild":59529,"upstreamProject":"node-test-commit-windows-fanned","upstreamUrl":"job/node-test-commit-windows-fanned/"},{"upstreamBuild":66977,"upstreamProject":"node-test-commit","upstreamUrl":"job/node-test-commit/"},{"upstreamBuild":55992,"upstreamProject":"node-test-pull-request","upstreamUrl":"job/node-test-pull-request/"},{"prId":"50758"}],"tap":{"ok":false,"name":"parallel/test-permission-fs-wildcard","id":578,"buffered":false,"tapError":null,"skip":false,"todo":false,"previous":null,"plan":null,"diag":{"severity":"fail","exitcode":1,"stack":"node:assert:126\n  throw new AssertionError(obj);\n  ^\n\nAssertionError [ERR_ASSERTION]: node:assert:408\n\n    throw err;\n\n    ^\n\n\n\nAssertionError [ERR_ASSERTION]: false == true\n\n    at [eval]:6:16\n\n    at runScriptInThisContext (node:internal/vm:144:10)\n\n    at node:internal/process/execution:109:14\n\n    at [eval]-wrapper:6:24\n\n    at runScript (node:internal/process/execution:92:62)\n\n    at evalScript (node:internal/process/execution:123:10)\n\n    at node:internal/main/eval_string:51:3 {\n\n  generatedMessage: true,\n\n  code: 'ERR_ASSERTION',\n\n  actual: false,\n\n  expected: true,\n\n  operator: '=='\n\n}\n\n\n\nNode.js v22.0.0-pre\n\n\n    at Object.<anonymous> (C:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\parallel\\test-permission-fs-wildcard.js:51:10)\n    at Module._compile (node:internal/modules/cjs/loader:1375:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1434:10)\n    at Module.load (node:internal/modules/cjs/loader:1206:32)\n    at Module._load (node:internal/modules/cjs/loader:1022:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:142:12)\n    at node:internal/main/run_main_module:28:49 {\n  generatedMessage: false,\n  code: 'ERR_ASSERTION',\n  actual: 1,\n  expected: 0,\n  operator: 'strictEqual'\n}\n\nNode.js v22.0.0-pre"},"time":556.002,"fullname":"parallel/test-permission-fs-wildcard","closingTestPoint":false}},{"testName":"parallel/test-permission-fs-write","timestamp":1701217871799,"status":"REGRESSION","jobName":"node-test-binary-windows-js-suites","config":"node-test-binary-windows-js-suites » 3,win2022-COMPILED_BY-vs2022 #24735","buildUrl":"https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=3,nodes=win2022-COMPILED_BY-vs2022/24735/","buildNumber":24735,"builtOn":"test-azure_msft-win2022_vs2019-x64-3","nodeVersion":"22.0.0","commitHash":"b2649a3bb95e6a322396f5c6e4a70a9e4a9c110e","callStack":[{"upstreamBuild":59529,"upstreamProject":"node-test-commit-windows-fanned","upstreamUrl":"job/node-test-commit-windows-fanned/"},{"upstreamBuild":66977,"upstreamProject":"node-test-commit","upstreamUrl":"job/node-test-commit/"},{"upstreamBuild":55992,"upstreamProject":"node-test-pull-request","upstreamUrl":"job/node-test-pull-request/"},{"prId":"50758"}],"tap":{"ok":false,"name":"parallel/test-permission-fs-write","id":580,"buffered":false,"tapError":null,"skip":false,"todo":false,"previous":null,"plan":null,"diag":{"severity":"fail","exitcode":1,"stack":"node:assert:126\n  throw new AssertionError(obj);\n  ^\n\nAssertionError [ERR_ASSERTION]: node:assert:408\n\n    throw err;\n\n    ^\n\n\n\nAssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:\n\n\n\n  assert.ok(!process.permission.has('fs.write', blockedFolder))\n\n\n\n    at Object.<anonymous> (C:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\fixtures\\permission\\fs-write.js:20:10)\n\n    at Module._compile (node:internal/modules/cjs/loader:1375:14)\n\n    at Module._extensions..js (node:internal/modules/cjs/loader:1434:10)\n\n    at Module.load (node:internal/modules/cjs/loader:1206:32)\n\n    at Module._load (node:internal/modules/cjs/loader:1022:12)\n\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:142:12)\n\n    at node:internal/main/run_main_module:28:49 {\n\n  generatedMessage: true,\n\n  code: 'ERR_ASSERTION',\n\n  actual: false,\n\n  expected: true,\n\n  operator: '=='\n\n}\n\n\n\nNode.js v22.0.0-pre\n\n\n    at Object.<anonymous> (C:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\parallel\\test-permission-fs-write.js:44:10)\n    at Module._compile (node:internal/modules/cjs/loader:1375:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1434:10)\n    at Module.load (node:internal/modules/cjs/loader:1206:32)\n    at Module._load (node:internal/modules/cjs/loader:1022:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:142:12)\n    at node:internal/main/run_main_module:28:49 {\n  generatedMessage: false,\n  code: 'ERR_ASSERTION',\n  actual: 1,\n  expected: 0,\n  operator: 'strictEqual'\n}\n\nNode.js v22.0.0-pre"},"time":616.06,"fullname":"parallel/test-permission-fs-write","closingTestPoint":false}},{"testName":"parallel/test-permission-fs-wildcard","timestamp":1701217871799,"status":"REGRESSION","jobName":"node-test-binary-windows-js-suites","config":"node-test-binary-windows-js-suites » 0,win11-arm64-COMPILED_BY-vs2022-arm64 #24735","buildUrl":"https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=0,nodes=win11-arm64-COMPILED_BY-vs2022-arm64/24735/","buildNumber":24735,"builtOn":"test-azure_msft-win11_vs2022-arm64-5","nodeVersion":"22.0.0","commitHash":"b2649a3bb95e6a322396f5c6e4a70a9e4a9c110e","callStack":[{"upstreamBuild":59529,"upstreamProject":"node-test-commit-windows-fanned","upstreamUrl":"job/node-test-commit-windows-fanned/"},{"upstreamBuild":66977,"upstreamProject":"node-test-commit","upstreamUrl":"job/node-test-commit/"},{"upstreamBuild":55992,"upstreamProject":"node-test-pull-request","upstreamUrl":"job/node-test-pull-request/"},{"prId":"50758"}],"tap":{"ok":false,"name":"parallel/test-permission-fs-wildcard","id":577,"buffered":false,"tapError":null,"skip":false,"todo":false,"previous":null,"plan":null,"diag":{"severity":"fail","exitcode":1,"stack":"node:assert:126\n  throw new AssertionError(obj);\n  ^\n\nAssertionError [ERR_ASSERTION]: node:assert:408\n\n    throw err;\n\n    ^\n\n\n\nAssertionError [ERR_ASSERTION]: false == true\n\n    at [eval]:6:16\n\n    at runScriptInThisContext (node:internal/vm:144:10)\n\n    at node:internal/process/execution:109:14\n\n    at [eval]-wrapper:6:24\n\n    at runScript (node:internal/process/execution:92:62)\n\n    at evalScript (node:internal/process/execution:123:10)\n\n    at node:internal/main/eval_string:51:3 {\n\n  generatedMessage: true,\n\n  code: 'ERR_ASSERTION',\n\n  actual: false,\n\n  expected: true,\n\n  operator: '=='\n\n}\n\n\n\nNode.js v22.0.0-pre\n\n\n    at Object.<anonymous> (d:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\parallel\\test-permission-fs-wildcard.js:51:10)\n    at Module._compile (node:internal/modules/cjs/loader:1375:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1434:10)\n    at Module.load (node:internal/modules/cjs/loader:1206:32)\n    at Module._load (node:internal/modules/cjs/loader:1022:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:142:12)\n    at node:internal/main/run_main_module:28:49 {\n  generatedMessage: false,\n  code: 'ERR_ASSERTION',\n  actual: 1,\n  expected: 0,\n  operator: 'strictEqual'\n}\n\nNode.js v22.0.0-pre"},"time":471.995,"fullname":"parallel/test-permission-fs-wildcard","closingTestPoint":false}},{"testName":"parallel/test-permission-fs-write","timestamp":1701217871799,"status":"REGRESSION","jobName":"node-test-binary-windows-js-suites","config":"node-test-binary-windows-js-suites » 0,win11-arm64-COMPILED_BY-vs2022-arm64 #24735","buildUrl":"https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=0,nodes=win11-arm64-COMPILED_BY-vs2022-arm64/24735/","buildNumber":24735,"builtOn":"test-azure_msft-win11_vs2022-arm64-5","nodeVersion":"22.0.0","commitHash":"b2649a3bb95e6a322396f5c6e4a70a9e4a9c110e","callStack":[{"upstreamBuild":59529,"upstreamProject":"node-test-commit-windows-fanned","upstreamUrl":"job/node-test-commit-windows-fanned/"},{"upstreamBuild":66977,"upstreamProject":"node-test-commit","upstreamUrl":"job/node-test-commit/"},{"upstreamBuild":55992,"upstreamProject":"node-test-pull-request","upstreamUrl":"job/node-test-pull-request/"},{"prId":"50758"}],"tap":{"ok":false,"name":"parallel/test-permission-fs-write","id":579,"buffered":false,"tapError":null,"skip":false,"todo":false,"previous":null,"plan":null,"diag":{"severity":"fail","exitcode":1,"stack":"node:assert:126\n  throw new AssertionError(obj);\n  ^\n\nAssertionError [ERR_ASSERTION]: node:assert:408\n\n    throw err;\n\n    ^\n\n\n\nAssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:\n\n\n\n  assert.ok(!process.permission.has('fs.write', blockedFolder))\n\n\n\n    at Object.<anonymous> (d:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\fixtures\\permission\\fs-write.js:20:10)\n\n    at Module._compile (node:internal/modules/cjs/loader:1375:14)\n\n    at Module._extensions..js (node:internal/modules/cjs/loader:1434:10)\n\n    at Module.load (node:internal/modules/cjs/loader:1206:32)\n\n    at Module._load (node:internal/modules/cjs/loader:1022:12)\n\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:142:12)\n\n    at node:internal/main/run_main_module:28:49 {\n\n  generatedMessage: true,\n\n  code: 'ERR_ASSERTION',\n\n  actual: false,\n\n  expected: true,\n\n  operator: '=='\n\n}\n\n\n\nNode.js v22.0.0-pre\n\n\n    at Object.<anonymous> (d:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\parallel\\test-permission-fs-write.js:44:10)\n    at Module._compile (node:internal/modules/cjs/loader:1375:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1434:10)\n    at Module.load (node:internal/modules/cjs/loader:1206:32)\n    at Module._load (node:internal/modules/cjs/loader:1022:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:142:12)\n    at node:internal/main/run_main_module:28:49 {\n  generatedMessage: false,\n  code: 'ERR_ASSERTION',\n  actual: 1,\n  expected: 0,\n  operator: 'strictEqual'\n}\n\nNode.js v22.0.0-pre"},"time":630.009,"fullname":"parallel/test-permission-fs-write","closingTestPoint":false}},{"testName":"parallel/test-permission-fs-read","timestamp":1701217871799,"status":"REGRESSION","jobName":"node-test-binary-windows-js-suites","config":"node-test-binary-windows-js-suites » 0,win2016-COMPILED_BY-vs2022-x86 #24735","buildUrl":"https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=0,nodes=win2016-COMPILED_BY-vs2022-x86/24735/","buildNumber":24735,"builtOn":"test-azure_msft-win2016_vs2017-x64-6","nodeVersion":"22.0.0","commitHash":"b2649a3bb95e6a322396f5c6e4a70a9e4a9c110e","callStack":[{"upstreamBuild":59529,"upstreamProject":"node-test-commit-windows-fanned","upstreamUrl":"job/node-test-commit-windows-fanned/"},{"upstreamBuild":66977,"upstreamProject":"node-test-commit","upstreamUrl":"job/node-test-commit/"},{"upstreamBuild":55992,"upstreamProject":"node-test-pull-request","upstreamUrl":"job/node-test-pull-request/"},{"prId":"50758"}],"tap":{"ok":false,"name":"parallel/test-permission-fs-read","id":575,"buffered":false,"tapError":null,"skip":false,"todo":false,"previous":null,"plan":null,"diag":{"severity":"fail","exitcode":1,"stack":"node:assert:126\n  throw new AssertionError(obj);\n  ^\n\nAssertionError [ERR_ASSERTION]: node:assert:126\n\n  throw new AssertionError(obj);\n\n  ^\n\n\n\nAssertionError [ERR_ASSERTION]: Missing expected exception.\n\n    at Object.<anonymous> (C:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\fixtures\\permission\\fs-read.js:16:10)\n\n    at Module._compile (node:internal/modules/cjs/loader:1375:14)\n\n    at Module._extensions..js (node:internal/modules/cjs/loader:1434:10)\n\n    at Module.load (node:internal/modules/cjs/loader:1206:32)\n\n    at Module._load (node:internal/modules/cjs/loader:1022:12)\n\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:142:12)\n\n    at node:internal/main/run_main_module:28:49 {\n\n  generatedMessage: false,\n\n  code: 'ERR_ASSERTION',\n\n  actual: undefined,\n\n  expected: [Function (anonymous)],\n\n  operator: 'throws'\n\n}\n\n\n\nNode.js v22.0.0-pre\n\n\n    at Object.<anonymous> (C:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\parallel\\test-permission-fs-read.js:42:10)\n    at Module._compile (node:internal/modules/cjs/loader:1375:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1434:10)\n    at Module.load (node:internal/modules/cjs/loader:1206:32)\n    at Module._load (node:internal/modules/cjs/loader:1022:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:142:12)\n    at node:internal/main/run_main_module:28:49 {\n  generatedMessage: false,\n  code: 'ERR_ASSERTION',\n  actual: 1,\n  expected: 0,\n  operator: 'strictEqual'\n}\n\nNode.js v22.0.0-pre"},"time":260.999,"fullname":"parallel/test-permission-fs-read","closingTestPoint":false}},{"testName":"parallel/test-permission-fs-symlink-target-write","timestamp":1701217871799,"status":"REGRESSION","jobName":"node-test-binary-windows-js-suites","config":"node-test-binary-windows-js-suites » 0,win2016-COMPILED_BY-vs2022-x86 #24735","buildUrl":"https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=0,nodes=win2016-COMPILED_BY-vs2022-x86/24735/","buildNumber":24735,"builtOn":"test-azure_msft-win2016_vs2017-x64-6","nodeVersion":"22.0.0","commitHash":"b2649a3bb95e6a322396f5c6e4a70a9e4a9c110e","callStack":[{"upstreamBuild":59529,"upstreamProject":"node-test-commit-windows-fanned","upstreamUrl":"job/node-test-commit-windows-fanned/"},{"upstreamBuild":66977,"upstreamProject":"node-test-commit","upstreamUrl":"job/node-test-commit/"},{"upstreamBuild":55992,"upstreamProject":"node-test-pull-request","upstreamUrl":"job/node-test-pull-request/"},{"prId":"50758"}],"tap":{"ok":false,"name":"parallel/test-permission-fs-symlink-target-write","id":577,"buffered":false,"tapError":null,"skip":false,"todo":false,"previous":null,"plan":null,"diag":{"severity":"fail","exitcode":1,"stack":"node:assert:126\n  throw new AssertionError(obj);\n  ^\n\nAssertionError [ERR_ASSERTION]: node:assert:408\n\n    throw err;\n\n    ^\n\n\n\nAssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:\n\n\n\n  assert.ok(!process.permission.has('fs.write', readOnlyFolder))\n\n\n\n    at Object.<anonymous> (C:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\fixtures\\permission\\fs-symlink-target-write.js:14:10)\n\n    at Module._compile (node:internal/modules/cjs/loader:1375:14)\n\n    at Module._extensions..js (node:internal/modules/cjs/loader:1434:10)\n\n    at Module.load (node:internal/modules/cjs/loader:1206:32)\n\n    at Module._load (node:internal/modules/cjs/loader:1022:12)\n\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:142:12)\n\n    at node:internal/main/run_main_module:28:49 {\n\n  generatedMessage: true,\n\n  code: 'ERR_ASSERTION',\n\n  actual: false,\n\n  expected: true,\n\n  operator: '=='\n\n}\n\n\n\nNode.js v22.0.0-pre\n\n\n    at Object.<anonymous> (C:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\parallel\\test-permission-fs-symlink-target-write.js:52:10)\n    at Module._compile (node:internal/modules/cjs/loader:1375:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1434:10)\n    at Module.load (node:internal/modules/cjs/loader:1206:32)\n    at Module._load (node:internal/modules/cjs/loader:1022:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:142:12)\n    at node:internal/main/run_main_module:28:49 {\n  generatedMessage: false,\n  code: 'ERR_ASSERTION',\n  actual: 1,\n  expected: 0,\n  operator: 'strictEqual'\n}\n\nNode.js v22.0.0-pre"},"time":379.999,"fullname":"parallel/test-permission-fs-symlink-target-write","closingTestPoint":false}},{"testName":"parallel/test-permission-fs-windows-path","timestamp":1701217871799,"status":"REGRESSION","jobName":"node-test-binary-windows-js-suites","config":"node-test-binary-windows-js-suites » 0,win2016-COMPILED_BY-vs2022-x86 #24735","buildUrl":"https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=0,nodes=win2016-COMPILED_BY-vs2022-x86/24735/","buildNumber":24735,"builtOn":"test-azure_msft-win2016_vs2017-x64-6","nodeVersion":"22.0.0","commitHash":"b2649a3bb95e6a322396f5c6e4a70a9e4a9c110e","callStack":[{"upstreamBuild":59529,"upstreamProject":"node-test-commit-windows-fanned","upstreamUrl":"job/node-test-commit-windows-fanned/"},{"upstreamBuild":66977,"upstreamProject":"node-test-commit","upstreamUrl":"job/node-test-commit/"},{"upstreamBuild":55992,"upstreamProject":"node-test-pull-request","upstreamUrl":"job/node-test-pull-request/"},{"prId":"50758"}],"tap":{"ok":false,"name":"parallel/test-permission-fs-windows-path","id":582,"buffered":false,"tapError":null,"skip":false,"todo":false,"previous":null,"plan":null,"diag":{"severity":"fail","exitcode":1,"stack":"node:assert:126\n  throw new AssertionError(obj);\n  ^\n\nAssertionError [ERR_ASSERTION]: (node:3580) ExperimentalWarning: Permission is an experimental feature\n(Use `node --trace-warnings ...` to show where the warning was created)\n\n    at Object.<anonymous> (C:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\parallel\\test-permission-fs-windows-path.js:28:10)\n    at Module._compile (node:internal/modules/cjs/loader:1375:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1434:10)\n    at Module.load (node:internal/modules/cjs/loader:1206:32)\n    at Module._load (node:internal/modules/cjs/loader:1022:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:142:12)\n    at node:internal/main/run_main_module:28:49 {\n  generatedMessage: false,\n  code: 'ERR_ASSERTION',\n  actual: 'true\\n',\n  expected: 'false\\n',\n  operator: 'strictEqual'\n}\n\nNode.js v22.0.0-pre"},"time":292.999,"fullname":"parallel/test-permission-fs-windows-path","closingTestPoint":false}},{"testName":"parallel/test-permission-fs-symlink","timestamp":1701217871799,"status":"REGRESSION","jobName":"node-test-binary-windows-js-suites","config":"node-test-binary-windows-js-suites » 1,win2019-COMPILED_BY-vs2022 #24735","buildUrl":"https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=1,nodes=win2019-COMPILED_BY-vs2022/24735/","buildNumber":24735,"builtOn":"test-rackspace-win2019_vs2019-x64-2","nodeVersion":"22.0.0","commitHash":"b2649a3bb95e6a322396f5c6e4a70a9e4a9c110e","callStack":[{"upstreamBuild":59529,"upstreamProject":"node-test-commit-windows-fanned","upstreamUrl":"job/node-test-commit-windows-fanned/"},{"upstreamBuild":66977,"upstreamProject":"node-test-commit","upstreamUrl":"job/node-test-commit/"},{"upstreamBuild":55992,"upstreamProject":"node-test-pull-request","upstreamUrl":"job/node-test-pull-request/"},{"prId":"50758"}],"tap":{"ok":false,"name":"parallel/test-permission-fs-symlink","id":582,"buffered":false,"tapError":null,"skip":false,"todo":false,"previous":null,"plan":null,"diag":{"severity":"fail","exitcode":1,"stack":"node:assert:126\n  throw new AssertionError(obj);\n  ^\n\nAssertionError [ERR_ASSERTION]: node:assert:408\n\n    throw err;\n\n    ^\n\n\n\nAssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:\n\n\n\n  assert.ok(!process.permission.has('fs.read', blockedFile))\n\n\n\n    at Object.<anonymous> (C:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\fixtures\\permission\\fs-symlink.js:15:10)\n\n    at Module._compile (node:internal/modules/cjs/loader:1375:14)\n\n    at Module._extensions..js (node:internal/modules/cjs/loader:1434:10)\n\n    at Module.load (node:internal/modules/cjs/loader:1206:32)\n\n    at Module._load (node:internal/modules/cjs/loader:1022:12)\n\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:142:12)\n\n    at node:internal/main/run_main_module:28:49 {\n\n  generatedMessage: true,\n\n  code: 'ERR_ASSERTION',\n\n  actual: false,\n\n  expected: true,\n\n  operator: '=='\n\n}\n\n\n\nNode.js v22.0.0-pre\n\n\n    at Object.<anonymous> (C:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\parallel\\test-permission-fs-symlink.js:53:10)\n    at Module._compile (node:internal/modules/cjs/loader:1375:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1434:10)\n    at Module.load (node:internal/modules/cjs/loader:1206:32)\n    at Module._load (node:internal/modules/cjs/loader:1022:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:142:12)\n    at node:internal/main/run_main_module:28:49 {\n  generatedMessage: false,\n  code: 'ERR_ASSERTION',\n  actual: 1,\n  expected: 0,\n  operator: 'strictEqual'\n}\n\nNode.js v22.0.0-pre"},"time":814.005,"fullname":"parallel/test-permission-fs-symlink","closingTestPoint":false}},{"testName":"parallel/test-permission-fs-read","timestamp":1701217871799,"status":"REGRESSION","jobName":"node-test-binary-windows-js-suites","config":"node-test-binary-windows-js-suites » 0,win2022-COMPILED_BY-vs2022 #24735","buildUrl":"https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=0,nodes=win2022-COMPILED_BY-vs2022/24735/","buildNumber":24735,"builtOn":"test-azure_msft-win2022_vs2019-x64-4","nodeVersion":"22.0.0","commitHash":"b2649a3bb95e6a322396f5c6e4a70a9e4a9c110e","callStack":[{"upstreamBuild":59529,"upstreamProject":"node-test-commit-windows-fanned","upstreamUrl":"job/node-test-commit-windows-fanned/"},{"upstreamBuild":66977,"upstreamProject":"node-test-commit","upstreamUrl":"job/node-test-commit/"},{"upstreamBuild":55992,"upstreamProject":"node-test-pull-request","upstreamUrl":"job/node-test-pull-request/"},{"prId":"50758"}],"tap":{"ok":false,"name":"parallel/test-permission-fs-read","id":575,"buffered":false,"tapError":null,"skip":false,"todo":false,"previous":null,"plan":null,"diag":{"severity":"fail","exitcode":1,"stack":"node:assert:126\n  throw new AssertionError(obj);\n  ^\n\nAssertionError [ERR_ASSERTION]: node:assert:126\n\n  throw new AssertionError(obj);\n\n  ^\n\n\n\nAssertionError [ERR_ASSERTION]: Missing expected exception.\n\n    at Object.<anonymous> (C:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\fixtures\\permission\\fs-read.js:16:10)\n\n    at Module._compile (node:internal/modules/cjs/loader:1375:14)\n\n    at Module._extensions..js (node:internal/modules/cjs/loader:1434:10)\n\n    at Module.load (node:internal/modules/cjs/loader:1206:32)\n\n    at Module._load (node:internal/modules/cjs/loader:1022:12)\n\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:142:12)\n\n    at node:internal/main/run_main_module:28:49 {\n\n  generatedMessage: false,\n\n  code: 'ERR_ASSERTION',\n\n  actual: undefined,\n\n  expected: [Function (anonymous)],\n\n  operator: 'throws'\n\n}\n\n\n\nNode.js v22.0.0-pre\n\n\n    at Object.<anonymous> (C:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\parallel\\test-permission-fs-read.js:42:10)\n    at Module._compile (node:internal/modules/cjs/loader:1375:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1434:10)\n    at Module.load (node:internal/modules/cjs/loader:1206:32)\n    at Module._load (node:internal/modules/cjs/loader:1022:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:142:12)\n    at node:internal/main/run_main_module:28:49 {\n  generatedMessage: false,\n  code: 'ERR_ASSERTION',\n  actual: 1,\n  expected: 0,\n  operator: 'strictEqual'\n}\n\nNode.js v22.0.0-pre"},"time":335.017,"fullname":"parallel/test-permission-fs-read","closingTestPoint":false}},{"testName":"parallel/test-permission-fs-symlink-target-write","timestamp":1701217871799,"status":"REGRESSION","jobName":"node-test-binary-windows-js-suites","config":"node-test-binary-windows-js-suites » 0,win2022-COMPILED_BY-vs2022 #24735","buildUrl":"https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=0,nodes=win2022-COMPILED_BY-vs2022/24735/","buildNumber":24735,"builtOn":"test-azure_msft-win2022_vs2019-x64-4","nodeVersion":"22.0.0","commitHash":"b2649a3bb95e6a322396f5c6e4a70a9e4a9c110e","callStack":[{"upstreamBuild":59529,"upstreamProject":"node-test-commit-windows-fanned","upstreamUrl":"job/node-test-commit-windows-fanned/"},{"upstreamBuild":66977,"upstreamProject":"node-test-commit","upstreamUrl":"job/node-test-commit/"},{"upstreamBuild":55992,"upstreamProject":"node-test-pull-request","upstreamUrl":"job/node-test-pull-request/"},{"prId":"50758"}],"tap":{"ok":false,"name":"parallel/test-permission-fs-symlink-target-write","id":576,"buffered":false,"tapError":null,"skip":false,"todo":false,"previous":null,"plan":null,"diag":{"severity":"fail","exitcode":1,"stack":"node:assert:126\n  throw new AssertionError(obj);\n  ^\n\nAssertionError [ERR_ASSERTION]: node:assert:408\n\n    throw err;\n\n    ^\n\n\n\nAssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:\n\n\n\n  assert.ok(!process.permission.has('fs.write', readOnlyFolder))\n\n\n\n    at Object.<anonymous> (C:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\fixtures\\permission\\fs-symlink-target-write.js:14:10)\n\n    at Module._compile (node:internal/modules/cjs/loader:1375:14)\n\n    at Module._extensions..js (node:internal/modules/cjs/loader:1434:10)\n\n    at Module.load (node:internal/modules/cjs/loader:1206:32)\n\n    at Module._load (node:internal/modules/cjs/loader:1022:12)\n\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:142:12)\n\n    at node:internal/main/run_main_module:28:49 {\n\n  generatedMessage: true,\n\n  code: 'ERR_ASSERTION',\n\n  actual: false,\n\n  expected: true,\n\n  operator: '=='\n\n}\n\n\n\nNode.js v22.0.0-pre\n\n\n    at Object.<anonymous> (C:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\parallel\\test-permission-fs-symlink-target-write.js:52:10)\n    at Module._compile (node:internal/modules/cjs/loader:1375:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1434:10)\n    at Module.load (node:internal/modules/cjs/loader:1206:32)\n    at Module._load (node:internal/modules/cjs/loader:1022:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:142:12)\n    at node:internal/main/run_main_module:28:49 {\n  generatedMessage: false,\n  code: 'ERR_ASSERTION',\n  actual: 1,\n  expected: 0,\n  operator: 'strictEqual'\n}\n\nNode.js v22.0.0-pre"},"time":481.024,"fullname":"parallel/test-permission-fs-symlink-target-write","closingTestPoint":false}},{"testName":"parallel/test-permission-fs-windows-path","timestamp":1701217871799,"status":"REGRESSION","jobName":"node-test-binary-windows-js-suites","config":"node-test-binary-windows-js-suites » 0,win2022-COMPILED_BY-vs2022 #24735","buildUrl":"https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=0,nodes=win2022-COMPILED_BY-vs2022/24735/","buildNumber":24735,"builtOn":"test-azure_msft-win2022_vs2019-x64-4","nodeVersion":"22.0.0","commitHash":"b2649a3bb95e6a322396f5c6e4a70a9e4a9c110e","callStack":[{"upstreamBuild":59529,"upstreamProject":"node-test-commit-windows-fanned","upstreamUrl":"job/node-test-commit-windows-fanned/"},{"upstreamBuild":66977,"upstreamProject":"node-test-commit","upstreamUrl":"job/node-test-commit/"},{"upstreamBuild":55992,"upstreamProject":"node-test-pull-request","upstreamUrl":"job/node-test-pull-request/"},{"prId":"50758"}],"tap":{"ok":false,"name":"parallel/test-permission-fs-windows-path","id":582,"buffered":false,"tapError":null,"skip":false,"todo":false,"previous":null,"plan":null,"diag":{"severity":"fail","exitcode":1,"stack":"node:assert:126\n  throw new AssertionError(obj);\n  ^\n\nAssertionError [ERR_ASSERTION]: (node:6524) ExperimentalWarning: Permission is an experimental feature\n(Use `node --trace-warnings ...` to show where the warning was created)\n\n    at Object.<anonymous> (C:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\parallel\\test-permission-fs-windows-path.js:28:10)\n    at Module._compile (node:internal/modules/cjs/loader:1375:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1434:10)\n    at Module.load (node:internal/modules/cjs/loader:1206:32)\n    at Module._load (node:internal/modules/cjs/loader:1022:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:142:12)\n    at node:internal/main/run_main_module:28:49 {\n  generatedMessage: false,\n  code: 'ERR_ASSERTION',\n  actual: 'true\\n',\n  expected: 'false\\n',\n  operator: 'strictEqual'\n}\n\nNode.js v22.0.0-pre"},"time":409.022,"fullname":"parallel/test-permission-fs-windows-path","closingTestPoint":false}},{"testName":"parallel/test-permission-fs-traversal-path","timestamp":1701217871799,"status":"REGRESSION","jobName":"node-test-binary-windows-js-suites","config":"node-test-binary-windows-js-suites » 3,win11-arm64-COMPILED_BY-vs2022-arm64 #24735","buildUrl":"https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=3,nodes=win11-arm64-COMPILED_BY-vs2022-arm64/24735/","buildNumber":24735,"builtOn":"test-azure_msft-win11_vs2022-arm64-3","nodeVersion":"22.0.0","commitHash":"b2649a3bb95e6a322396f5c6e4a70a9e4a9c110e","callStack":[{"upstreamBuild":59529,"upstreamProject":"node-test-commit-windows-fanned","upstreamUrl":"job/node-test-commit-windows-fanned/"},{"upstreamBuild":66977,"upstreamProject":"node-test-commit","upstreamUrl":"job/node-test-commit/"},{"upstreamBuild":55992,"upstreamProject":"node-test-pull-request","upstreamUrl":"job/node-test-pull-request/"},{"prId":"50758"}],"tap":{"ok":false,"name":"parallel/test-permission-fs-traversal-path","id":590,"buffered":false,"tapError":null,"skip":false,"todo":false,"previous":null,"plan":null,"diag":{"severity":"fail","exitcode":1,"stack":"node:assert:126\n  throw new AssertionError(obj);\n  ^\n\nAssertionError [ERR_ASSERTION]: node:assert:408\n\n    throw err;\n\n    ^\n\n\n\nAssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:\n\n\n\n  assert.ok(!process.permission.has('fs.read', blockedFolder))\n\n\n\n    at Object.<anonymous> (d:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\fixtures\\permission\\fs-traversal.js:23:10)\n\n    at Module._compile (node:internal/modules/cjs/loader:1375:14)\n\n    at Module._extensions..js (node:internal/modules/cjs/loader:1434:10)\n\n    at Module.load (node:internal/modules/cjs/loader:1206:32)\n\n    at Module._load (node:internal/modules/cjs/loader:1022:12)\n\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:142:12)\n\n    at node:internal/main/run_main_module:28:49 {\n\n  generatedMessage: true,\n\n  code: 'ERR_ASSERTION',\n\n  actual: false,\n\n  expected: true,\n\n  operator: '=='\n\n}\n\n\n\nNode.js v22.0.0-pre\n\n\n    at Object.<anonymous> (d:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\parallel\\test-permission-fs-traversal-path.js:46:10)\n    at Module._compile (node:internal/modules/cjs/loader:1375:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1434:10)\n    at Module.load (node:internal/modules/cjs/loader:1206:32)\n    at Module._load (node:internal/modules/cjs/loader:1022:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:142:12)\n    at node:internal/main/run_main_module:28:49 {\n  generatedMessage: false,\n  code: 'ERR_ASSERTION',\n  actual: 1,\n  expected: 0,\n  operator: 'strictEqual'\n}\n\nNode.js v22.0.0-pre"},"time":934,"fullname":"parallel/test-permission-fs-traversal-path","closingTestPoint":false}},{"testName":"parallel/test-permission-fs-symlink","timestamp":1701217871799,"status":"REGRESSION","jobName":"node-test-binary-windows-js-suites","config":"node-test-binary-windows-js-suites » 1,win2022-COMPILED_BY-vs2022 #24735","buildUrl":"https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=1,nodes=win2022-COMPILED_BY-vs2022/24735/","buildNumber":24735,"builtOn":"test-azure_msft-win2022_vs2019-x64-2","nodeVersion":"22.0.0","commitHash":"b2649a3bb95e6a322396f5c6e4a70a9e4a9c110e","callStack":[{"upstreamBuild":59529,"upstreamProject":"node-test-commit-windows-fanned","upstreamUrl":"job/node-test-commit-windows-fanned/"},{"upstreamBuild":66977,"upstreamProject":"node-test-commit","upstreamUrl":"job/node-test-commit/"},{"upstreamBuild":55992,"upstreamProject":"node-test-pull-request","upstreamUrl":"job/node-test-pull-request/"},{"prId":"50758"}],"tap":{"ok":false,"name":"parallel/test-permission-fs-symlink","id":581,"buffered":false,"tapError":null,"skip":false,"todo":false,"previous":null,"plan":null,"diag":{"severity":"fail","exitcode":1,"stack":"node:assert:126\n  throw new AssertionError(obj);\n  ^\n\nAssertionError [ERR_ASSERTION]: node:assert:408\n\n    throw err;\n\n    ^\n\n\n\nAssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:\n\n\n\n  assert.ok(!process.permission.has('fs.read', blockedFile))\n\n\n\n    at Object.<anonymous> (C:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\fixtures\\permission\\fs-symlink.js:15:10)\n\n    at Module._compile (node:internal/modules/cjs/loader:1375:14)\n\n    at Module._extensions..js (node:internal/modules/cjs/loader:1434:10)\n\n    at Module.load (node:internal/modules/cjs/loader:1206:32)\n\n    at Module._load (node:internal/modules/cjs/loader:1022:12)\n\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:142:12)\n\n    at node:internal/main/run_main_module:28:49 {\n\n  generatedMessage: true,\n\n  code: 'ERR_ASSERTION',\n\n  actual: false,\n\n  expected: true,\n\n  operator: '=='\n\n}\n\n\n\nNode.js v22.0.0-pre\n\n\n    at Object.<anonymous> (C:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\parallel\\test-permission-fs-symlink.js:53:10)\n    at Module._compile (node:internal/modules/cjs/loader:1375:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1434:10)\n    at Module.load (node:internal/modules/cjs/loader:1206:32)\n    at Module._load (node:internal/modules/cjs/loader:1022:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:142:12)\n    at node:internal/main/run_main_module:28:49 {\n  generatedMessage: false,\n  code: 'ERR_ASSERTION',\n  actual: 1,\n  expected: 0,\n  operator: 'strictEqual'\n}\n\nNode.js v22.0.0-pre"},"time":610.006,"fullname":"parallel/test-permission-fs-symlink","closingTestPoint":false}},{"testName":"es-module/test-cjs-legacyMainResolve-permission","timestamp":1701217871799,"status":"FAILED","jobName":"node-test-binary-windows-js-suites","config":"node-test-binary-windows-js-suites » 2,win2022-COMPILED_BY-vs2022 #24735","buildUrl":"https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=2,nodes=win2022-COMPILED_BY-vs2022/24735/","buildNumber":24735,"builtOn":"test-rackspace-win2022_vs2022-x64-6","nodeVersion":"22.0.0","commitHash":"b2649a3bb95e6a322396f5c6e4a70a9e4a9c110e","callStack":[{"upstreamBuild":59529,"upstreamProject":"node-test-commit-windows-fanned","upstreamUrl":"job/node-test-commit-windows-fanned/"},{"upstreamBuild":66977,"upstreamProject":"node-test-commit","upstreamUrl":"job/node-test-commit/"},{"upstreamBuild":55992,"upstreamProject":"node-test-pull-request","upstreamUrl":"job/node-test-pull-request/"},{"prId":"50758"}],"tap":{"ok":false,"name":"es-module/test-cjs-legacyMainResolve-permission","id":27,"buffered":false,"tapError":null,"skip":false,"todo":false,"previous":null,"plan":null,"diag":{"severity":"fail","exitcode":1,"stack":"TAP version 13\n# Subtest: legacyMainResolve\n    # Subtest: should ensure permission model when resolving by packageConfig.main\n    not ok 1 - should ensure permission model when resolving by packageConfig.main\n      ---\n      duration_ms: 243.0152\n      location: 'C:\\\\workspace\\\\node-test-binary-windows-js-suites\\\\node\\\\test\\\\es-module\\\\test-cjs-legacyMainResolve-permission.js:19:3'\n      failureType: 'testCodeFailure'\n      error: |-\n        node:assert:126\n          throw new AssertionError(obj);\n          ^\n        \n        AssertionError [ERR_ASSERTION]: Missing expected exception.\n            at [eval]:19:20\n            at runScriptInThisContext (node:internal/vm:144:10)\n            at node:internal/process/execution:109:14\n            at [eval]-wrapper:6:24\n            at runScript (node:internal/process/execution:92:62)\n            at evalScript (node:internal/process/execution:123:10)\n            at node:internal/main/eval_string:51:3 {\n          generatedMessage: false,\n          code: 'ERR_ASSERTION',\n          actual: undefined,\n          expected: {\n            code: 'ERR_ACCESS_DENIED',\n            resource: 'C:\\\\workspace\\\\node-test-binary-windows-js-suites\\\\node\\\\test\\\\fixtures\\\\es-modules\\\\legacy-main-resolver\\\\index-js\\\\index'\n          },\n          operator: 'throws'\n        }\n        \n        Node.js v22.0.0-pre\n        \n      code: 'ERR_ASSERTION'\n      name: 'AssertionError'\n      expected: 0\n      actual: 1\n      operator: 'strictEqual'\n      stack: |-\n        [eval]:19:20\n        runScriptInThisContext (node:internal/vm:144:10)\n        node:internal/process/execution:109:14\n        [eval]-wrapper:6:24\n        runScript (node:internal/process/execution:92:62)\n        evalScript (node:internal/process/execution:123:10)\n        node:internal/main/eval_string:51:3 {\n        TestContext.<anonymous> (C:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\es-module\\test-cjs-legacyMainResolve-permission.js:75:14)\n        Test.runInAsyncScope (node:async_hooks:206:9)\n        Test.run (node:internal/test_runner/test:631:25)\n        Test.start (node:internal/test_runner/test:542:17)\n        node:internal/test_runner/test:946:71\n        node:internal/per_context/primordials:488:82\n        new Promise (<anonymous>)\n        new SafePromise (node:internal/per_context/primordials:456:29)\n        node:internal/per_context/primordials:488:9\n        Array.map (<anonymous>)\n      ...\n    # Subtest: should ensure permission model when resolving by packageJsonUrl\n    not ok 2 - should ensure permission model when resolving by packageJsonUrl\n      ---\n      duration_ms: 215.572\n      location: 'C:\\\\workspace\\\\node-test-binary-windows-js-suites\\\\node\\\\test\\\\es-module\\\\test-cjs-legacyMainResolve-permission.js:79:3'\n      failureType: 'testCodeFailure'\n      error: |-\n        node:assert:126\n          throw new AssertionError(obj);\n          ^\n        \n        AssertionError [ERR_ASSERTION]: Missing expected exception.\n            at [eval]:20:20\n            at runScriptInThisContext (node:internal/vm:144:10)\n            at node:internal/process/execution:109:14\n            at [eval]-wrapper:6:24\n            at runScript (node:internal/process/execution:92:62)\n            at evalScript (node:internal/process/execution:123:10)\n            at node:internal/main/eval_string:51:3 {\n          generatedMessage: false,\n          code: 'ERR_ASSERTION',\n          actual: undefined,\n          expected: {\n            code: 'ERR_ACCESS_DENIED',\n            resource: 'C:\\\\workspace\\\\node-test-binary-windows-js-suites\\\\node\\\\test\\\\fixtures\\\\es-modules\\\\legacy-main-resolver\\\\index-json\\\\index.json'\n          },\n          operator: 'throws'\n        }\n        \n        Node.js v22.0.0-pre\n        \n      code: 'ERR_ASSERTION'\n      name: 'AssertionError'\n      expected: 0\n      actual: 1\n      operator: 'strictEqual'\n      stack: |-\n        [eval]:20:20\n        runScriptInThisContext (node:internal/vm:144:10)\n        node:internal/process/execution:109:14\n        [eval]-wrapper:6:24\n        runScript (node:internal/process/execution:92:62)\n        evalScript (node:internal/process/execution:123:10)\n        node:internal/main/eval_string:51:3 {\n        TestContext.<anonymous> (C:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\es-module\\test-cjs-legacyMainResolve-permission.js:133:14)\n        Test.runInAsyncScope (node:async_hooks:206:9)\n        Test.run (node:internal/test_runner/test:631:25)\n        Suite.processPendingSubtests (node:internal/test_runner/test:374:18)\n        Test.postRun (node:internal/test_runner/test:715:19)\n        Test.run (node:internal/test_runner/test:673:12)\n        process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n        async Promise.all (index 0)\n        async Suite.run (node:internal/test_runner/test:948:7)\n        async startSubtest (node:internal/test_runner/harness:216:3)\n      ...\n    1..2\nnot ok 1 - legacyMainResolve\n  ---\n  duration_ms: 464.4997\n  type: 'suite'\n  location: 'C:\\\\workspace\\\\node-test-binary-windows-js-suites\\\\node\\\\test\\\\es-module\\\\test-cjs-legacyMainResolve-permission.js:18:1'\n  failureType: 'subtestsFailed'\n  error: '2 subtests failed'\n  code: 'ERR_TEST_FAILURE'\n  ...\n1..1\n# tests 2\n# suites 1\n# pass 0\n# fail 2\n# cancelled 0\n# skipped 0\n# todo 0\n# duration_ms 496.2886\n(node:7516) ExperimentalWarning: Permission is an experimental feature\n(Use `node --trace-warnings ...` to show where the warning was created)\n(node:7516) SecurityWarning: The flag --allow-child-process must be used with extreme caution. It could invalidate the permission model."},"time":932.015,"fullname":"es-module/test-cjs-legacyMainResolve-permission","closingTestPoint":false}},{"testName":"parallel/test-permission-fs-traversal-path","timestamp":1701217871799,"status":"REGRESSION","jobName":"node-test-binary-windows-js-suites","config":"node-test-binary-windows-js-suites » 2,win2022-COMPILED_BY-vs2022 #24735","buildUrl":"https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=2,nodes=win2022-COMPILED_BY-vs2022/24735/","buildNumber":24735,"builtOn":"test-rackspace-win2022_vs2022-x64-6","nodeVersion":"22.0.0","commitHash":"b2649a3bb95e6a322396f5c6e4a70a9e4a9c110e","callStack":[{"upstreamBuild":59529,"upstreamProject":"node-test-commit-windows-fanned","upstreamUrl":"job/node-test-commit-windows-fanned/"},{"upstreamBuild":66977,"upstreamProject":"node-test-commit","upstreamUrl":"job/node-test-commit/"},{"upstreamBuild":55992,"upstreamProject":"node-test-pull-request","upstreamUrl":"job/node-test-pull-request/"},{"prId":"50758"}],"tap":{"ok":false,"name":"parallel/test-permission-fs-traversal-path","id":589,"buffered":false,"tapError":null,"skip":false,"todo":false,"previous":null,"plan":null,"diag":{"severity":"fail","exitcode":1,"stack":"node:assert:126\n  throw new AssertionError(obj);\n  ^\n\nAssertionError [ERR_ASSERTION]: node:assert:408\n\n    throw err;\n\n    ^\n\n\n\nAssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:\n\n\n\n  assert.ok(!process.permission.has('fs.read', blockedFolder))\n\n\n\n    at Object.<anonymous> (C:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\fixtures\\permission\\fs-traversal.js:23:10)\n\n    at Module._compile (node:internal/modules/cjs/loader:1375:14)\n\n    at Module._extensions..js (node:internal/modules/cjs/loader:1434:10)\n\n    at Module.load (node:internal/modules/cjs/loader:1206:32)\n\n    at Module._load (node:internal/modules/cjs/loader:1022:12)\n\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:142:12)\n\n    at node:internal/main/run_main_module:28:49 {\n\n  generatedMessage: true,\n\n  code: 'ERR_ASSERTION',\n\n  actual: false,\n\n  expected: true,\n\n  operator: '=='\n\n}\n\n\n\nNode.js v22.0.0-pre\n\n\n    at Object.<anonymous> (C:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\parallel\\test-permission-fs-traversal-path.js:46:10)\n    at Module._compile (node:internal/modules/cjs/loader:1375:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1434:10)\n    at Module.load (node:internal/modules/cjs/loader:1206:32)\n    at Module._load (node:internal/modules/cjs/loader:1022:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:142:12)\n    at node:internal/main/run_main_module:28:49 {\n  generatedMessage: false,\n  code: 'ERR_ASSERTION',\n  actual: 1,\n  expected: 0,\n  operator: 'strictEqual'\n}\n\nNode.js v22.0.0-pre"},"time":840.016,"fullname":"parallel/test-permission-fs-traversal-path","closingTestPoint":false}},{"testName":"es-module/test-cjs-legacyMainResolve-permission","timestamp":1701217871799,"status":"FAILED","jobName":"node-test-binary-windows-js-suites","config":"node-test-binary-windows-js-suites » 2,win2016-COMPILED_BY-vs2022-x86 #24735","buildUrl":"https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=2,nodes=win2016-COMPILED_BY-vs2022-x86/24735/","buildNumber":24735,"builtOn":"test-azure_msft-win2016_vs2017-x64-1","nodeVersion":"22.0.0","commitHash":"b2649a3bb95e6a322396f5c6e4a70a9e4a9c110e","callStack":[{"upstreamBuild":59529,"upstreamProject":"node-test-commit-windows-fanned","upstreamUrl":"job/node-test-commit-windows-fanned/"},{"upstreamBuild":66977,"upstreamProject":"node-test-commit","upstreamUrl":"job/node-test-commit/"},{"upstreamBuild":55992,"upstreamProject":"node-test-pull-request","upstreamUrl":"job/node-test-pull-request/"},{"prId":"50758"}],"tap":{"ok":false,"name":"es-module/test-cjs-legacyMainResolve-permission","id":24,"buffered":false,"tapError":null,"skip":false,"todo":false,"previous":null,"plan":null,"diag":{"severity":"fail","exitcode":1,"stack":"TAP version 13\n# Subtest: legacyMainResolve\n    # Subtest: should ensure permission model when resolving by packageConfig.main\n    not ok 1 - should ensure permission model when resolving by packageConfig.main\n      ---\n      duration_ms: 158.4023\n      location: 'C:\\\\workspace\\\\node-test-binary-windows-js-suites\\\\node\\\\test\\\\es-module\\\\test-cjs-legacyMainResolve-permission.js:19:3'\n      failureType: 'testCodeFailure'\n      error: |-\n        node:assert:126\n          throw new AssertionError(obj);\n          ^\n        \n        AssertionError [ERR_ASSERTION]: Missing expected exception.\n            at [eval]:19:20\n            at runScriptInThisContext (node:internal/vm:144:10)\n            at node:internal/process/execution:109:14\n            at [eval]-wrapper:6:24\n            at runScript (node:internal/process/execution:92:62)\n            at evalScript (node:internal/process/execution:123:10)\n            at node:internal/main/eval_string:51:3 {\n          generatedMessage: false,\n          code: 'ERR_ASSERTION',\n          actual: undefined,\n          expected: {\n            code: 'ERR_ACCESS_DENIED',\n            resource: 'C:\\\\workspace\\\\node-test-binary-windows-js-suites\\\\node\\\\test\\\\fixtures\\\\es-modules\\\\legacy-main-resolver\\\\index-js\\\\index'\n          },\n          operator: 'throws'\n        }\n        \n        Node.js v22.0.0-pre\n        \n      code: 'ERR_ASSERTION'\n      name: 'AssertionError'\n      expected: 0\n      actual: 1\n      operator: 'strictEqual'\n      stack: |-\n        [eval]:19:20\n        runScriptInThisContext (node:internal/vm:144:10)\n        node:internal/process/execution:109:14\n        [eval]-wrapper:6:24\n        runScript (node:internal/process/execution:92:62)\n        evalScript (node:internal/process/execution:123:10)\n        node:internal/main/eval_string:51:3 {\n        TestContext.<anonymous> (C:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\es-module\\test-cjs-legacyMainResolve-permission.js:75:14)\n        Test.runInAsyncScope (node:async_hooks:206:9)\n        Test.run (node:internal/test_runner/test:631:25)\n        Test.start (node:internal/test_runner/test:542:17)\n        node:internal/test_runner/test:946:71\n        node:internal/per_context/primordials:488:82\n        new Promise (<anonymous>)\n        new SafePromise (node:internal/per_context/primordials:456:29)\n        node:internal/per_context/primordials:488:9\n        Array.map (<anonymous>)\n      ...\n    # Subtest: should ensure permission model when resolving by packageJsonUrl\n    not ok 2 - should ensure permission model when resolving by packageJsonUrl\n      ---\n      duration_ms: 121.1542\n      location: 'C:\\\\workspace\\\\node-test-binary-windows-js-suites\\\\node\\\\test\\\\es-module\\\\test-cjs-legacyMainResolve-permission.js:79:3'\n      failureType: 'testCodeFailure'\n      error: |-\n        node:assert:126\n          throw new AssertionError(obj);\n          ^\n        \n        AssertionError [ERR_ASSERTION]: Missing expected exception.\n            at [eval]:20:20\n            at runScriptInThisContext (node:internal/vm:144:10)\n            at node:internal/process/execution:109:14\n            at [eval]-wrapper:6:24\n            at runScript (node:internal/process/execution:92:62)\n            at evalScript (node:internal/process/execution:123:10)\n            at node:internal/main/eval_string:51:3 {\n          generatedMessage: false,\n          code: 'ERR_ASSERTION',\n          actual: undefined,\n          expected: {\n            code: 'ERR_ACCESS_DENIED',\n            resource: 'C:\\\\workspace\\\\node-test-binary-windows-js-suites\\\\node\\\\test\\\\fixtures\\\\es-modules\\\\legacy-main-resolver\\\\index-json\\\\index.json'\n          },\n          operator: 'throws'\n        }\n        \n        Node.js v22.0.0-pre\n        \n      code: 'ERR_ASSERTION'\n      name: 'AssertionError'\n      expected: 0\n      actual: 1\n      operator: 'strictEqual'\n      stack: |-\n        [eval]:20:20\n        runScriptInThisContext (node:internal/vm:144:10)\n        node:internal/process/execution:109:14\n        [eval]-wrapper:6:24\n        runScript (node:internal/process/execution:92:62)\n        evalScript (node:internal/process/execution:123:10)\n        node:internal/main/eval_string:51:3 {\n        TestContext.<anonymous> (C:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\es-module\\test-cjs-legacyMainResolve-permission.js:133:14)\n        Test.runInAsyncScope (node:async_hooks:206:9)\n        Test.run (node:internal/test_runner/test:631:25)\n        Suite.processPendingSubtests (node:internal/test_runner/test:374:18)\n        Test.postRun (node:internal/test_runner/test:715:19)\n        Test.run (node:internal/test_runner/test:673:12)\n        process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n        async Promise.all (index 0)\n        async Suite.run (node:internal/test_runner/test:948:7)\n        async startSubtest (node:internal/test_runner/harness:216:3)\n      ...\n    1..2\nnot ok 1 - legacyMainResolve\n  ---\n  duration_ms: 282.4823\n  type: 'suite'\n  location: 'C:\\\\workspace\\\\node-test-binary-windows-js-suites\\\\node\\\\test\\\\es-module\\\\test-cjs-legacyMainResolve-permission.js:18:1'\n  failureType: 'subtestsFailed'\n  error: '2 subtests failed'\n  code: 'ERR_TEST_FAILURE'\n  ...\n1..1\n# tests 2\n# suites 1\n# pass 0\n# fail 2\n# cancelled 0\n# skipped 0\n# todo 0\n# duration_ms 299.1263\n(node:4248) ExperimentalWarning: Permission is an experimental feature\n(Use `node --trace-warnings ...` to show where the warning was created)\n(node:4248) SecurityWarning: The flag --allow-child-process must be used with extreme caution. It could invalidate the permission model."},"time":525.006,"fullname":"es-module/test-cjs-legacyMainResolve-permission","closingTestPoint":false}},{"testName":"parallel/test-permission-fs-traversal-path","timestamp":1701217871799,"status":"REGRESSION","jobName":"node-test-binary-windows-js-suites","config":"node-test-binary-windows-js-suites » 2,win2016-COMPILED_BY-vs2022-x86 #24735","buildUrl":"https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=2,nodes=win2016-COMPILED_BY-vs2022-x86/24735/","buildNumber":24735,"builtOn":"test-azure_msft-win2016_vs2017-x64-1","nodeVersion":"22.0.0","commitHash":"b2649a3bb95e6a322396f5c6e4a70a9e4a9c110e","callStack":[{"upstreamBuild":59529,"upstreamProject":"node-test-commit-windows-fanned","upstreamUrl":"job/node-test-commit-windows-fanned/"},{"upstreamBuild":66977,"upstreamProject":"node-test-commit","upstreamUrl":"job/node-test-commit/"},{"upstreamBuild":55992,"upstreamProject":"node-test-pull-request","upstreamUrl":"job/node-test-pull-request/"},{"prId":"50758"}],"tap":{"ok":false,"name":"parallel/test-permission-fs-traversal-path","id":581,"buffered":false,"tapError":null,"skip":false,"todo":false,"previous":null,"plan":null,"diag":{"severity":"fail","exitcode":1,"stack":"node:assert:126\n  throw new AssertionError(obj);\n  ^\n\nAssertionError [ERR_ASSERTION]: node:assert:408\n\n    throw err;\n\n    ^\n\n\n\nAssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:\n\n\n\n  assert.ok(!process.permission.has('fs.read', blockedFolder))\n\n\n\n    at Object.<anonymous> (C:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\fixtures\\permission\\fs-traversal.js:23:10)\n\n    at Module._compile (node:internal/modules/cjs/loader:1375:14)\n\n    at Module._extensions..js (node:internal/modules/cjs/loader:1434:10)\n\n    at Module.load (node:internal/modules/cjs/loader:1206:32)\n\n    at Module._load (node:internal/modules/cjs/loader:1022:12)\n\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:142:12)\n\n    at node:internal/main/run_main_module:28:49 {\n\n  generatedMessage: true,\n\n  code: 'ERR_ASSERTION',\n\n  actual: false,\n\n  expected: true,\n\n  operator: '=='\n\n}\n\n\n\nNode.js v22.0.0-pre\n\n\n    at Object.<anonymous> (C:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\parallel\\test-permission-fs-traversal-path.js:46:10)\n    at Module._compile (node:internal/modules/cjs/loader:1375:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1434:10)\n    at Module.load (node:internal/modules/cjs/loader:1206:32)\n    at Module._load (node:internal/modules/cjs/loader:1022:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:142:12)\n    at node:internal/main/run_main_module:28:49 {\n  generatedMessage: false,\n  code: 'ERR_ASSERTION',\n  actual: 1,\n  expected: 0,\n  operator: 'strictEqual'\n}\n\nNode.js v22.0.0-pre"},"time":373.004,"fullname":"parallel/test-permission-fs-traversal-path","closingTestPoint":false}},{"testName":"parallel/test-permission-fs-read","timestamp":1701217871799,"status":"REGRESSION","jobName":"node-test-binary-windows-js-suites","config":"node-test-binary-windows-js-suites » 0,win2019-COMPILED_BY-vs2022 #24735","buildUrl":"https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=0,nodes=win2019-COMPILED_BY-vs2022/24735/","buildNumber":24735,"builtOn":"test-rackspace-win2019_vs2019-x64-4","nodeVersion":"22.0.0","commitHash":"b2649a3bb95e6a322396f5c6e4a70a9e4a9c110e","callStack":[{"upstreamBuild":59529,"upstreamProject":"node-test-commit-windows-fanned","upstreamUrl":"job/node-test-commit-windows-fanned/"},{"upstreamBuild":66977,"upstreamProject":"node-test-commit","upstreamUrl":"job/node-test-commit/"},{"upstreamBuild":55992,"upstreamProject":"node-test-pull-request","upstreamUrl":"job/node-test-pull-request/"},{"prId":"50758"}],"tap":{"ok":false,"name":"parallel/test-permission-fs-read","id":576,"buffered":false,"tapError":null,"skip":false,"todo":false,"previous":null,"plan":null,"diag":{"severity":"fail","exitcode":1,"stack":"node:assert:126\n  throw new AssertionError(obj);\n  ^\n\nAssertionError [ERR_ASSERTION]: node:assert:126\n\n  throw new AssertionError(obj);\n\n  ^\n\n\n\nAssertionError [ERR_ASSERTION]: Missing expected exception.\n\n    at Object.<anonymous> (C:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\fixtures\\permission\\fs-read.js:16:10)\n\n    at Module._compile (node:internal/modules/cjs/loader:1375:14)\n\n    at Module._extensions..js (node:internal/modules/cjs/loader:1434:10)\n\n    at Module.load (node:internal/modules/cjs/loader:1206:32)\n\n    at Module._load (node:internal/modules/cjs/loader:1022:12)\n\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:142:12)\n\n    at node:internal/main/run_main_module:28:49 {\n\n  generatedMessage: false,\n\n  code: 'ERR_ASSERTION',\n\n  actual: undefined,\n\n  expected: [Function (anonymous)],\n\n  operator: 'throws'\n\n}\n\n\n\nNode.js v22.0.0-pre\n\n\n    at Object.<anonymous> (C:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\parallel\\test-permission-fs-read.js:42:10)\n    at Module._compile (node:internal/modules/cjs/loader:1375:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1434:10)\n    at Module.load (node:internal/modules/cjs/loader:1206:32)\n    at Module._load (node:internal/modules/cjs/loader:1022:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:142:12)\n    at node:internal/main/run_main_module:28:49 {\n  generatedMessage: false,\n  code: 'ERR_ASSERTION',\n  actual: 1,\n  expected: 0,\n  operator: 'strictEqual'\n}\n\nNode.js v22.0.0-pre"},"time":1092.032,"fullname":"parallel/test-permission-fs-read","closingTestPoint":false}},{"testName":"parallel/test-permission-fs-windows-path","timestamp":1701217871799,"status":"REGRESSION","jobName":"node-test-binary-windows-js-suites","config":"node-test-binary-windows-js-suites » 0,win2019-COMPILED_BY-vs2022 #24735","buildUrl":"https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=0,nodes=win2019-COMPILED_BY-vs2022/24735/","buildNumber":24735,"builtOn":"test-rackspace-win2019_vs2019-x64-4","nodeVersion":"22.0.0","commitHash":"b2649a3bb95e6a322396f5c6e4a70a9e4a9c110e","callStack":[{"upstreamBuild":59529,"upstreamProject":"node-test-commit-windows-fanned","upstreamUrl":"job/node-test-commit-windows-fanned/"},{"upstreamBuild":66977,"upstreamProject":"node-test-commit","upstreamUrl":"job/node-test-commit/"},{"upstreamBuild":55992,"upstreamProject":"node-test-pull-request","upstreamUrl":"job/node-test-pull-request/"},{"prId":"50758"}],"tap":{"ok":false,"name":"parallel/test-permission-fs-windows-path","id":579,"buffered":false,"tapError":null,"skip":false,"todo":false,"previous":null,"plan":null,"diag":{"severity":"fail","exitcode":1,"stack":"node:assert:126\n  throw new AssertionError(obj);\n  ^\n\nAssertionError [ERR_ASSERTION]: (node:6176) ExperimentalWarning: Permission is an experimental feature\n(Use `node --trace-warnings ...` to show where the warning was created)\n\n    at Object.<anonymous> (C:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\parallel\\test-permission-fs-windows-path.js:28:10)\n    at Module._compile (node:internal/modules/cjs/loader:1375:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1434:10)\n    at Module.load (node:internal/modules/cjs/loader:1206:32)\n    at Module._load (node:internal/modules/cjs/loader:1022:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:142:12)\n    at node:internal/main/run_main_module:28:49 {\n  generatedMessage: false,\n  code: 'ERR_ASSERTION',\n  actual: 'true\\n',\n  expected: 'false\\n',\n  operator: 'strictEqual'\n}\n\nNode.js v22.0.0-pre"},"time":1400.088,"fullname":"parallel/test-permission-fs-windows-path","closingTestPoint":false}},{"testName":"parallel/test-permission-fs-symlink-target-write","timestamp":1701217871799,"status":"REGRESSION","jobName":"node-test-binary-windows-js-suites","config":"node-test-binary-windows-js-suites » 0,win2019-COMPILED_BY-vs2022 #24735","buildUrl":"https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=0,nodes=win2019-COMPILED_BY-vs2022/24735/","buildNumber":24735,"builtOn":"test-rackspace-win2019_vs2019-x64-4","nodeVersion":"22.0.0","commitHash":"b2649a3bb95e6a322396f5c6e4a70a9e4a9c110e","callStack":[{"upstreamBuild":59529,"upstreamProject":"node-test-commit-windows-fanned","upstreamUrl":"job/node-test-commit-windows-fanned/"},{"upstreamBuild":66977,"upstreamProject":"node-test-commit","upstreamUrl":"job/node-test-commit/"},{"upstreamBuild":55992,"upstreamProject":"node-test-pull-request","upstreamUrl":"job/node-test-pull-request/"},{"prId":"50758"}],"tap":{"ok":false,"name":"parallel/test-permission-fs-symlink-target-write","id":582,"buffered":false,"tapError":null,"skip":false,"todo":false,"previous":null,"plan":null,"diag":{"severity":"fail","exitcode":1,"stack":"node:assert:126\n  throw new AssertionError(obj);\n  ^\n\nAssertionError [ERR_ASSERTION]: node:assert:408\n\n    throw err;\n\n    ^\n\n\n\nAssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:\n\n\n\n  assert.ok(!process.permission.has('fs.write', readOnlyFolder))\n\n\n\n    at Object.<anonymous> (C:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\fixtures\\permission\\fs-symlink-target-write.js:14:10)\n\n    at Module._compile (node:internal/modules/cjs/loader:1375:14)\n\n    at Module._extensions..js (node:internal/modules/cjs/loader:1434:10)\n\n    at Module.load (node:internal/modules/cjs/loader:1206:32)\n\n    at Module._load (node:internal/modules/cjs/loader:1022:12)\n\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:142:12)\n\n    at node:internal/main/run_main_module:28:49 {\n\n  generatedMessage: true,\n\n  code: 'ERR_ASSERTION',\n\n  actual: false,\n\n  expected: true,\n\n  operator: '=='\n\n}\n\n\n\nNode.js v22.0.0-pre\n\n\n    at Object.<anonymous> (C:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\parallel\\test-permission-fs-symlink-target-write.js:52:10)\n    at Module._compile (node:internal/modules/cjs/loader:1375:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1434:10)\n    at Module.load (node:internal/modules/cjs/loader:1206:32)\n    at Module._load (node:internal/modules/cjs/loader:1022:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:142:12)\n    at node:internal/main/run_main_module:28:49 {\n  generatedMessage: false,\n  code: 'ERR_ASSERTION',\n  actual: 1,\n  expected: 0,\n  operator: 'strictEqual'\n}\n\nNode.js v22.0.0-pre"},"time":1635.079,"fullname":"parallel/test-permission-fs-symlink-target-write","closingTestPoint":false}},{"testName":"parallel/test-permission-fs-read","timestamp":1701217871799,"status":"REGRESSION","jobName":"node-test-binary-windows-js-suites","config":"node-test-binary-windows-js-suites » 1,win11-arm64-COMPILED_BY-vs2022-arm64 #24735","buildUrl":"https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=1,nodes=win11-arm64-COMPILED_BY-vs2022-arm64/24735/","buildNumber":24735,"builtOn":"test-azure_msft-win11_vs2022-arm64-6","nodeVersion":"22.0.0","commitHash":"b2649a3bb95e6a322396f5c6e4a70a9e4a9c110e","callStack":[{"upstreamBuild":59529,"upstreamProject":"node-test-commit-windows-fanned","upstreamUrl":"job/node-test-commit-windows-fanned/"},{"upstreamBuild":66977,"upstreamProject":"node-test-commit","upstreamUrl":"job/node-test-commit/"},{"upstreamBuild":55992,"upstreamProject":"node-test-pull-request","upstreamUrl":"job/node-test-pull-request/"},{"prId":"50758"}],"tap":{"ok":false,"name":"parallel/test-permission-fs-read","id":575,"buffered":false,"tapError":null,"skip":false,"todo":false,"previous":null,"plan":null,"diag":{"severity":"fail","exitcode":1,"stack":"node:assert:126\n  throw new AssertionError(obj);\n  ^\n\nAssertionError [ERR_ASSERTION]: node:assert:126\n\n  throw new AssertionError(obj);\n\n  ^\n\n\n\nAssertionError [ERR_ASSERTION]: Missing expected exception.\n\n    at Object.<anonymous> (d:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\fixtures\\permission\\fs-read.js:16:10)\n\n    at Module._compile (node:internal/modules/cjs/loader:1375:14)\n\n    at Module._extensions..js (node:internal/modules/cjs/loader:1434:10)\n\n    at Module.load (node:internal/modules/cjs/loader:1206:32)\n\n    at Module._load (node:internal/modules/cjs/loader:1022:12)\n\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:142:12)\n\n    at node:internal/main/run_main_module:28:49 {\n\n  generatedMessage: false,\n\n  code: 'ERR_ASSERTION',\n\n  actual: undefined,\n\n  expected: [Function (anonymous)],\n\n  operator: 'throws'\n\n}\n\n\n\nNode.js v22.0.0-pre\n\n\n    at Object.<anonymous> (d:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\parallel\\test-permission-fs-read.js:42:10)\n    at Module._compile (node:internal/modules/cjs/loader:1375:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1434:10)\n    at Module.load (node:internal/modules/cjs/loader:1206:32)\n    at Module._load (node:internal/modules/cjs/loader:1022:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:142:12)\n    at node:internal/main/run_main_module:28:49 {\n  generatedMessage: false,\n  code: 'ERR_ASSERTION',\n  actual: 1,\n  expected: 0,\n  operator: 'strictEqual'\n}\n\nNode.js v22.0.0-pre"},"time":873.983,"fullname":"parallel/test-permission-fs-read","closingTestPoint":false}},{"testName":"parallel/test-permission-fs-symlink-target-write","timestamp":1701217871799,"status":"REGRESSION","jobName":"node-test-binary-windows-js-suites","config":"node-test-binary-windows-js-suites » 1,win11-arm64-COMPILED_BY-vs2022-arm64 #24735","buildUrl":"https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=1,nodes=win11-arm64-COMPILED_BY-vs2022-arm64/24735/","buildNumber":24735,"builtOn":"test-azure_msft-win11_vs2022-arm64-6","nodeVersion":"22.0.0","commitHash":"b2649a3bb95e6a322396f5c6e4a70a9e4a9c110e","callStack":[{"upstreamBuild":59529,"upstreamProject":"node-test-commit-windows-fanned","upstreamUrl":"job/node-test-commit-windows-fanned/"},{"upstreamBuild":66977,"upstreamProject":"node-test-commit","upstreamUrl":"job/node-test-commit/"},{"upstreamBuild":55992,"upstreamProject":"node-test-pull-request","upstreamUrl":"job/node-test-pull-request/"},{"prId":"50758"}],"tap":{"ok":false,"name":"parallel/test-permission-fs-symlink-target-write","id":577,"buffered":false,"tapError":null,"skip":false,"todo":false,"previous":null,"plan":null,"diag":{"severity":"fail","exitcode":1,"stack":"node:assert:126\n  throw new AssertionError(obj);\n  ^\n\nAssertionError [ERR_ASSERTION]: node:assert:408\n\n    throw err;\n\n    ^\n\n\n\nAssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:\n\n\n\n  assert.ok(!process.permission.has('fs.write', readOnlyFolder))\n\n\n\n    at Object.<anonymous> (d:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\fixtures\\permission\\fs-symlink-target-write.js:14:10)\n\n    at Module._compile (node:internal/modules/cjs/loader:1375:14)\n\n    at Module._extensions..js (node:internal/modules/cjs/loader:1434:10)\n\n    at Module.load (node:internal/modules/cjs/loader:1206:32)\n\n    at Module._load (node:internal/modules/cjs/loader:1022:12)\n\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:142:12)\n\n    at node:internal/main/run_main_module:28:49 {\n\n  generatedMessage: true,\n\n  code: 'ERR_ASSERTION',\n\n  actual: false,\n\n  expected: true,\n\n  operator: '=='\n\n}\n\n\n\nNode.js v22.0.0-pre\n\n\n    at Object.<anonymous> (d:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\parallel\\test-permission-fs-symlink-target-write.js:52:10)\n    at Module._compile (node:internal/modules/cjs/loader:1375:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1434:10)\n    at Module.load (node:internal/modules/cjs/loader:1206:32)\n    at Module._load (node:internal/modules/cjs/loader:1022:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:142:12)\n    at node:internal/main/run_main_module:28:49 {\n  generatedMessage: false,\n  code: 'ERR_ASSERTION',\n  actual: 1,\n  expected: 0,\n  operator: 'strictEqual'\n}\n\nNode.js v22.0.0-pre"},"time":739.999,"fullname":"parallel/test-permission-fs-symlink-target-write","closingTestPoint":false}},{"testName":"parallel/test-cli-permission-deny-fs","timestamp":1701217871799,"status":"REGRESSION","jobName":"node-test-binary-windows-js-suites","config":"node-test-binary-windows-js-suites » 3,win2016-COMPILED_BY-vs2022-x86 #24735","buildUrl":"https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=3,nodes=win2016-COMPILED_BY-vs2022-x86/24735/","buildNumber":24735,"builtOn":"test-azure_msft-win2016_vs2017-x64-4","nodeVersion":"22.0.0","commitHash":"b2649a3bb95e6a322396f5c6e4a70a9e4a9c110e","callStack":[{"upstreamBuild":59529,"upstreamProject":"node-test-commit-windows-fanned","upstreamUrl":"job/node-test-commit-windows-fanned/"},{"upstreamBuild":66977,"upstreamProject":"node-test-commit","upstreamUrl":"job/node-test-commit/"},{"upstreamBuild":55992,"upstreamProject":"node-test-pull-request","upstreamUrl":"job/node-test-pull-request/"},{"prId":"50758"}],"tap":{"ok":false,"name":"parallel/test-cli-permission-deny-fs","id":127,"buffered":false,"tapError":null,"skip":false,"todo":false,"previous":null,"plan":null,"diag":{"severity":"fail","exitcode":1,"stack":"node:assert:1035\n    throw err;\n    ^\n\nAssertionError [ERR_ASSERTION]: The input did not match the regular expression /resource: '.*?[\\\\/](?:etc|passwd)'/. Input:\n\n'(node:4492) ExperimentalWarning: Permission is an experimental feature\\n' +\n  '(Use `node --trace-warnings ...` to show where the warning was created)\\n'\n\n    at Object.<anonymous> (C:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\parallel\\test-cli-permission-deny-fs.js:150:10)\n    at Module._compile (node:internal/modules/cjs/loader:1375:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1434:10)\n    at Module.load (node:internal/modules/cjs/loader:1206:32)\n    at Module._load (node:internal/modules/cjs/loader:1022:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:142:12)\n    at node:internal/main/run_main_module:28:49 {\n  generatedMessage: true,\n  code: 'ERR_ASSERTION',\n  actual: '(node:4492) ExperimentalWarning: Permission is an experimental feature\\n' +\n    '(Use `node --trace-warnings ...` to show where the warning was created)\\n',\n  expected: /resource: '.*?[\\\\/](?:etc|passwd)'/,\n  operator: 'match'\n}\n\nNode.js v22.0.0-pre"},"time":1982.2,"fullname":"parallel/test-cli-permission-deny-fs","closingTestPoint":false}},{"testName":"parallel/test-permission-fs-wildcard","timestamp":1701217871799,"status":"REGRESSION","jobName":"node-test-binary-windows-js-suites","config":"node-test-binary-windows-js-suites » 3,win2016-COMPILED_BY-vs2022-x86 #24735","buildUrl":"https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=3,nodes=win2016-COMPILED_BY-vs2022-x86/24735/","buildNumber":24735,"builtOn":"test-azure_msft-win2016_vs2017-x64-4","nodeVersion":"22.0.0","commitHash":"b2649a3bb95e6a322396f5c6e4a70a9e4a9c110e","callStack":[{"upstreamBuild":59529,"upstreamProject":"node-test-commit-windows-fanned","upstreamUrl":"job/node-test-commit-windows-fanned/"},{"upstreamBuild":66977,"upstreamProject":"node-test-commit","upstreamUrl":"job/node-test-commit/"},{"upstreamBuild":55992,"upstreamProject":"node-test-pull-request","upstreamUrl":"job/node-test-pull-request/"},{"prId":"50758"}],"tap":{"ok":false,"name":"parallel/test-permission-fs-wildcard","id":578,"buffered":false,"tapError":null,"skip":false,"todo":false,"previous":null,"plan":null,"diag":{"severity":"fail","exitcode":1,"stack":"node:assert:126\n  throw new AssertionError(obj);\n  ^\n\nAssertionError [ERR_ASSERTION]: node:assert:408\n\n    throw err;\n\n    ^\n\n\n\nAssertionError [ERR_ASSERTION]: false == true\n\n    at [eval]:6:16\n\n    at runScriptInThisContext (node:internal/vm:144:10)\n\n    at node:internal/process/execution:109:14\n\n    at [eval]-wrapper:6:24\n\n    at runScript (node:internal/process/execution:92:62)\n\n    at evalScript (node:internal/process/execution:123:10)\n\n    at node:internal/main/eval_string:51:3 {\n\n  generatedMessage: true,\n\n  code: 'ERR_ASSERTION',\n\n  actual: false,\n\n  expected: true,\n\n  operator: '=='\n\n}\n\n\n\nNode.js v22.0.0-pre\n\n\n    at Object.<anonymous> (C:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\parallel\\test-permission-fs-wildcard.js:51:10)\n    at Module._compile (node:internal/modules/cjs/loader:1375:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1434:10)\n    at Module.load (node:internal/modules/cjs/loader:1206:32)\n    at Module._load (node:internal/modules/cjs/loader:1022:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:142:12)\n    at node:internal/main/run_main_module:28:49 {\n  generatedMessage: false,\n  code: 'ERR_ASSERTION',\n  actual: 1,\n  expected: 0,\n  operator: 'strictEqual'\n}\n\nNode.js v22.0.0-pre"},"time":274.019,"fullname":"parallel/test-permission-fs-wildcard","closingTestPoint":false}},{"testName":"parallel/test-permission-fs-write","timestamp":1701217871799,"status":"REGRESSION","jobName":"node-test-binary-windows-js-suites","config":"node-test-binary-windows-js-suites » 3,win2016-COMPILED_BY-vs2022-x86 #24735","buildUrl":"https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=3,nodes=win2016-COMPILED_BY-vs2022-x86/24735/","buildNumber":24735,"builtOn":"test-azure_msft-win2016_vs2017-x64-4","nodeVersion":"22.0.0","commitHash":"b2649a3bb95e6a322396f5c6e4a70a9e4a9c110e","callStack":[{"upstreamBuild":59529,"upstreamProject":"node-test-commit-windows-fanned","upstreamUrl":"job/node-test-commit-windows-fanned/"},{"upstreamBuild":66977,"upstreamProject":"node-test-commit","upstreamUrl":"job/node-test-commit/"},{"upstreamBuild":55992,"upstreamProject":"node-test-pull-request","upstreamUrl":"job/node-test-pull-request/"},{"prId":"50758"}],"tap":{"ok":false,"name":"parallel/test-permission-fs-write","id":579,"buffered":false,"tapError":null,"skip":false,"todo":false,"previous":null,"plan":null,"diag":{"severity":"fail","exitcode":1,"stack":"node:assert:126\n  throw new AssertionError(obj);\n  ^\n\nAssertionError [ERR_ASSERTION]: node:assert:408\n\n    throw err;\n\n    ^\n\n\n\nAssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:\n\n\n\n  assert.ok(!process.permission.has('fs.write', blockedFolder))\n\n\n\n    at Object.<anonymous> (C:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\fixtures\\permission\\fs-write.js:20:10)\n\n    at Module._compile (node:internal/modules/cjs/loader:1375:14)\n\n    at Module._extensions..js (node:internal/modules/cjs/loader:1434:10)\n\n    at Module.load (node:internal/modules/cjs/loader:1206:32)\n\n    at Module._load (node:internal/modules/cjs/loader:1022:12)\n\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:142:12)\n\n    at node:internal/main/run_main_module:28:49 {\n\n  generatedMessage: true,\n\n  code: 'ERR_ASSERTION',\n\n  actual: false,\n\n  expected: true,\n\n  operator: '=='\n\n}\n\n\n\nNode.js v22.0.0-pre\n\n\n    at Object.<anonymous> (C:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\parallel\\test-permission-fs-write.js:44:10)\n    at Module._compile (node:internal/modules/cjs/loader:1375:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1434:10)\n    at Module.load (node:internal/modules/cjs/loader:1206:32)\n    at Module._load (node:internal/modules/cjs/loader:1022:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:142:12)\n    at node:internal/main/run_main_module:28:49 {\n  generatedMessage: false,\n  code: 'ERR_ASSERTION',\n  actual: 1,\n  expected: 0,\n  operator: 'strictEqual'\n}\n\nNode.js v22.0.0-pre"},"time":336.019,"fullname":"parallel/test-permission-fs-write","closingTestPoint":false}},{"testName":"parallel/test-runner-watch-mode","timestamp":1701217871799,"status":"FAILED","jobName":"node-test-binary-windows-js-suites","config":"node-test-binary-windows-js-suites » 3,win2016-COMPILED_BY-vs2022-x86 #24735","buildUrl":"https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=3,nodes=win2016-COMPILED_BY-vs2022-x86/24735/","buildNumber":24735,"builtOn":"test-azure_msft-win2016_vs2017-x64-4","nodeVersion":"22.0.0","commitHash":"b2649a3bb95e6a322396f5c6e4a70a9e4a9c110e","callStack":[{"upstreamBuild":59529,"upstreamProject":"node-test-commit-windows-fanned","upstreamUrl":"job/node-test-commit-windows-fanned/"},{"upstreamBuild":66977,"upstreamProject":"node-test-commit","upstreamUrl":"job/node-test-commit/"},{"upstreamBuild":55992,"upstreamProject":"node-test-pull-request","upstreamUrl":"job/node-test-pull-request/"},{"prId":"50758"}],"tap":{"ok":false,"name":"parallel/test-runner-watch-mode","id":721,"buffered":false,"tapError":null,"skip":false,"todo":": Fix flaky test","previous":null,"plan":null,"diag":{"severity":"flaky","exitcode":1,"stack":"TAP version 13\n# Subtest: test runner watch mode\n    # Subtest: should run tests repeatedly\n    ok 1 - should run tests repeatedly\n      ---\n      duration_ms: 1879.3878\n      ...\n    # Subtest: should run tests with dependency repeatedly\n    ok 2 - should run tests with dependency repeatedly\n      ---\n      duration_ms: 1512.3627\n      ...\n    # Subtest: should run tests with ESM dependency\n    ok 3 - should run tests with ESM dependency\n      ---\n      duration_ms: 1611.466\n      ...\n    # Subtest: should support running tests without a file\n    ok 4 - should support running tests without a file\n      ---\n      duration_ms: 1542.61\n      ...\n    1..4\nok 1 - test runner watch mode\n  ---\n  duration_ms: 6549.7179\n  type: 'suite'\n  ...\n1..1\n# tests 4\n# suites 1\n# pass 4\n# fail 0\n# cancelled 0\n# skipped 0\n# todo 0\n# duration_ms 6576.5222\nCan't clean tmpdir: C:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\.tmp.652\nFiles blocking: [ 'dependency.js', 'dependency.mjs', 'test.js' ]\n\nC:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\common\\tmpdir.js:69\n    throw e;\n    ^\n\nError: EBUSY: resource busy or locked, rmdir '\\\\?\\C:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\.tmp.652'\n    at rmdirSync (node:fs:1221:10)\n    at _rmdirSync (node:internal/fs/rimraf:235:5)\n    at rimrafSync (node:internal/fs/rimraf:193:7)\n    at Object.rmSync (node:fs:1270:10)\n    at rmSync (C:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\common\\tmpdir.js:20:8)\n    at onexit (C:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\common\\tmpdir.js:54:5)\n    at process.<anonymous> (C:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\common\\tmpdir.js:43:14)\n    at process.emit (node:events:531:35) {\n  errno: -4082,\n  syscall: 'rmdir',\n  code: 'EBUSY',\n  path: '\\\\\\\\?\\\\C:\\\\workspace\\\\node-test-binary-windows-js-suites\\\\node\\\\test\\\\.tmp.652'\n}\n\nNode.js v22.0.0-pre"},"time":7838.557,"fullname":"parallel/test-runner-watch-mode","closingTestPoint":false}},{"testName":"es-module/test-cjs-legacyMainResolve-permission","timestamp":1701217871799,"status":"FAILED","jobName":"node-test-binary-windows-js-suites","config":"node-test-binary-windows-js-suites » 2,win2019-COMPILED_BY-vs2022 #24735","buildUrl":"https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=2,nodes=win2019-COMPILED_BY-vs2022/24735/","buildNumber":24735,"builtOn":"test-rackspace-win2019_vs2019-x64-1","nodeVersion":"22.0.0","commitHash":"b2649a3bb95e6a322396f5c6e4a70a9e4a9c110e","callStack":[{"upstreamBuild":59529,"upstreamProject":"node-test-commit-windows-fanned","upstreamUrl":"job/node-test-commit-windows-fanned/"},{"upstreamBuild":66977,"upstreamProject":"node-test-commit","upstreamUrl":"job/node-test-commit/"},{"upstreamBuild":55992,"upstreamProject":"node-test-pull-request","upstreamUrl":"job/node-test-pull-request/"},{"prId":"50758"}],"tap":{"ok":false,"name":"es-module/test-cjs-legacyMainResolve-permission","id":27,"buffered":false,"tapError":null,"skip":false,"todo":false,"previous":null,"plan":null,"diag":{"severity":"fail","exitcode":1,"stack":"TAP version 13\n# Subtest: legacyMainResolve\n    # Subtest: should ensure permission model when resolving by packageConfig.main\n    not ok 1 - should ensure permission model when resolving by packageConfig.main\n      ---\n      duration_ms: 268.912\n      location: 'C:\\\\workspace\\\\node-test-binary-windows-js-suites\\\\node\\\\test\\\\es-module\\\\test-cjs-legacyMainResolve-permission.js:19:3'\n      failureType: 'testCodeFailure'\n      error: |-\n        node:assert:126\n          throw new AssertionError(obj);\n          ^\n        \n        AssertionError [ERR_ASSERTION]: Missing expected exception.\n            at [eval]:19:20\n            at runScriptInThisContext (node:internal/vm:144:10)\n            at node:internal/process/execution:109:14\n            at [eval]-wrapper:6:24\n            at runScript (node:internal/process/execution:92:62)\n            at evalScript (node:internal/process/execution:123:10)\n            at node:internal/main/eval_string:51:3 {\n          generatedMessage: false,\n          code: 'ERR_ASSERTION',\n          actual: undefined,\n          expected: {\n            code: 'ERR_ACCESS_DENIED',\n            resource: 'C:\\\\workspace\\\\node-test-binary-windows-js-suites\\\\node\\\\test\\\\fixtures\\\\es-modules\\\\legacy-main-resolver\\\\index-js\\\\index'\n          },\n          operator: 'throws'\n        }\n        \n        Node.js v22.0.0-pre\n        \n      code: 'ERR_ASSERTION'\n      name: 'AssertionError'\n      expected: 0\n      actual: 1\n      operator: 'strictEqual'\n      stack: |-\n        [eval]:19:20\n        runScriptInThisContext (node:internal/vm:144:10)\n        node:internal/process/execution:109:14\n        [eval]-wrapper:6:24\n        runScript (node:internal/process/execution:92:62)\n        evalScript (node:internal/process/execution:123:10)\n        node:internal/main/eval_string:51:3 {\n        TestContext.<anonymous> (C:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\es-module\\test-cjs-legacyMainResolve-permission.js:75:14)\n        Test.runInAsyncScope (node:async_hooks:206:9)\n        Test.run (node:internal/test_runner/test:631:25)\n        Test.start (node:internal/test_runner/test:542:17)\n        node:internal/test_runner/test:946:71\n        node:internal/per_context/primordials:488:82\n        new Promise (<anonymous>)\n        new SafePromise (node:internal/per_context/primordials:456:29)\n        node:internal/per_context/primordials:488:9\n        Array.map (<anonymous>)\n      ...\n    # Subtest: should ensure permission model when resolving by packageJsonUrl\n    not ok 2 - should ensure permission model when resolving by packageJsonUrl\n      ---\n      duration_ms: 221.485\n      location: 'C:\\\\workspace\\\\node-test-binary-windows-js-suites\\\\node\\\\test\\\\es-module\\\\test-cjs-legacyMainResolve-permission.js:79:3'\n      failureType: 'testCodeFailure'\n      error: |-\n        node:assert:126\n          throw new AssertionError(obj);\n          ^\n        \n        AssertionError [ERR_ASSERTION]: Missing expected exception.\n            at [eval]:20:20\n            at runScriptInThisContext (node:internal/vm:144:10)\n            at node:internal/process/execution:109:14\n            at [eval]-wrapper:6:24\n            at runScript (node:internal/process/execution:92:62)\n            at evalScript (node:internal/process/execution:123:10)\n            at node:internal/main/eval_string:51:3 {\n          generatedMessage: false,\n          code: 'ERR_ASSERTION',\n          actual: undefined,\n          expected: {\n            code: 'ERR_ACCESS_DENIED',\n            resource: 'C:\\\\workspace\\\\node-test-binary-windows-js-suites\\\\node\\\\test\\\\fixtures\\\\es-modules\\\\legacy-main-resolver\\\\index-json\\\\index.json'\n          },\n          operator: 'throws'\n        }\n        \n        Node.js v22.0.0-pre\n        \n      code: 'ERR_ASSERTION'\n      name: 'AssertionError'\n      expected: 0\n      actual: 1\n      operator: 'strictEqual'\n      stack: |-\n        [eval]:20:20\n        runScriptInThisContext (node:internal/vm:144:10)\n        node:internal/process/execution:109:14\n        [eval]-wrapper:6:24\n        runScript (node:internal/process/execution:92:62)\n        evalScript (node:internal/process/execution:123:10)\n        node:internal/main/eval_string:51:3 {\n        TestContext.<anonymous> (C:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\es-module\\test-cjs-legacyMainResolve-permission.js:133:14)\n        Test.runInAsyncScope (node:async_hooks:206:9)\n        Test.run (node:internal/test_runner/test:631:25)\n        Suite.processPendingSubtests (node:internal/test_runner/test:374:18)\n        Test.postRun (node:internal/test_runner/test:715:19)\n        Test.run (node:internal/test_runner/test:673:12)\n        process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n        async Promise.all (index 0)\n        async Suite.run (node:internal/test_runner/test:948:7)\n        async startSubtest (node:internal/test_runner/harness:216:3)\n      ...\n    1..2\nnot ok 1 - legacyMainResolve\n  ---\n  duration_ms: 496.5943\n  type: 'suite'\n  location: 'C:\\\\workspace\\\\node-test-binary-windows-js-suites\\\\node\\\\test\\\\es-module\\\\test-cjs-legacyMainResolve-permission.js:18:1'\n  failureType: 'subtestsFailed'\n  error: '2 subtests failed'\n  code: 'ERR_TEST_FAILURE'\n  ...\n1..1\n# tests 2\n# suites 1\n# pass 0\n# fail 2\n# cancelled 0\n# skipped 0\n# todo 0\n# duration_ms 529.5865\n(node:2744) ExperimentalWarning: Permission is an experimental feature\n(Use `node --trace-warnings ...` to show where the warning was created)\n(node:2744) SecurityWarning: The flag --allow-child-process must be used with extreme caution. It could invalidate the permission model."},"time":919.996,"fullname":"es-module/test-cjs-legacyMainResolve-permission","closingTestPoint":false}},{"testName":"parallel/test-permission-fs-traversal-path","timestamp":1701217871799,"status":"REGRESSION","jobName":"node-test-binary-windows-js-suites","config":"node-test-binary-windows-js-suites » 2,win2019-COMPILED_BY-vs2022 #24735","buildUrl":"https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=2,nodes=win2019-COMPILED_BY-vs2022/24735/","buildNumber":24735,"builtOn":"test-rackspace-win2019_vs2019-x64-1","nodeVersion":"22.0.0","commitHash":"b2649a3bb95e6a322396f5c6e4a70a9e4a9c110e","callStack":[{"upstreamBuild":59529,"upstreamProject":"node-test-commit-windows-fanned","upstreamUrl":"job/node-test-commit-windows-fanned/"},{"upstreamBuild":66977,"upstreamProject":"node-test-commit","upstreamUrl":"job/node-test-commit/"},{"upstreamBuild":55992,"upstreamProject":"node-test-pull-request","upstreamUrl":"job/node-test-pull-request/"},{"prId":"50758"}],"tap":{"ok":false,"name":"parallel/test-permission-fs-traversal-path","id":587,"buffered":false,"tapError":null,"skip":false,"todo":false,"previous":null,"plan":null,"diag":{"severity":"fail","exitcode":1,"stack":"node:assert:126\n  throw new AssertionError(obj);\n  ^\n\nAssertionError [ERR_ASSERTION]: node:assert:408\n\n    throw err;\n\n    ^\n\n\n\nAssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:\n\n\n\n  assert.ok(!process.permission.has('fs.read', blockedFolder))\n\n\n\n    at Object.<anonymous> (C:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\fixtures\\permission\\fs-traversal.js:23:10)\n\n    at Module._compile (node:internal/modules/cjs/loader:1375:14)\n\n    at Module._extensions..js (node:internal/modules/cjs/loader:1434:10)\n\n    at Module.load (node:internal/modules/cjs/loader:1206:32)\n\n    at Module._load (node:internal/modules/cjs/loader:1022:12)\n\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:142:12)\n\n    at node:internal/main/run_main_module:28:49 {\n\n  generatedMessage: true,\n\n  code: 'ERR_ASSERTION',\n\n  actual: false,\n\n  expected: true,\n\n  operator: '=='\n\n}\n\n\n\nNode.js v22.0.0-pre\n\n\n    at Object.<anonymous> (C:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\parallel\\test-permission-fs-traversal-path.js:46:10)\n    at Module._compile (node:internal/modules/cjs/loader:1375:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1434:10)\n    at Module.load (node:internal/modules/cjs/loader:1206:32)\n    at Module._load (node:internal/modules/cjs/loader:1022:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:142:12)\n    at node:internal/main/run_main_module:28:49 {\n  generatedMessage: false,\n  code: 'ERR_ASSERTION',\n  actual: 1,\n  expected: 0,\n  operator: 'strictEqual'\n}\n\nNode.js v22.0.0-pre"},"time":745.071,"fullname":"parallel/test-permission-fs-traversal-path","closingTestPoint":false}},{"testName":"es-module/test-cjs-legacyMainResolve-permission","timestamp":1701217871799,"status":"FAILED","jobName":"node-test-binary-windows-js-suites","config":"node-test-binary-windows-js-suites » 2,win11-arm64-COMPILED_BY-vs2022-arm64 #24735","buildUrl":"https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=2,nodes=win11-arm64-COMPILED_BY-vs2022-arm64/24735/","buildNumber":24735,"builtOn":"test-azure_msft-win11_vs2022-arm64-4","nodeVersion":"22.0.0","commitHash":"b2649a3bb95e6a322396f5c6e4a70a9e4a9c110e","callStack":[{"upstreamBuild":59529,"upstreamProject":"node-test-commit-windows-fanned","upstreamUrl":"job/node-test-commit-windows-fanned/"},{"upstreamBuild":66977,"upstreamProject":"node-test-commit","upstreamUrl":"job/node-test-commit/"},{"upstreamBuild":55992,"upstreamProject":"node-test-pull-request","upstreamUrl":"job/node-test-pull-request/"},{"prId":"50758"}],"tap":{"ok":false,"name":"es-module/test-cjs-legacyMainResolve-permission","id":34,"buffered":false,"tapError":null,"skip":false,"todo":false,"previous":null,"plan":null,"diag":{"severity":"fail","exitcode":1,"stack":"TAP version 13\n# Subtest: legacyMainResolve\n    # Subtest: should ensure permission model when resolving by packageConfig.main\n    not ok 1 - should ensure permission model when resolving by packageConfig.main\n      ---\n      duration_ms: 377.615\n      location: 'd:\\\\workspace\\\\node-test-binary-windows-js-suites\\\\node\\\\test\\\\es-module\\\\test-cjs-legacyMainResolve-permission.js:19:3'\n      failureType: 'testCodeFailure'\n      error: |-\n        node:assert:126\n          throw new AssertionError(obj);\n          ^\n        \n        AssertionError [ERR_ASSERTION]: Missing expected exception.\n            at [eval]:19:20\n            at runScriptInThisContext (node:internal/vm:144:10)\n            at node:internal/process/execution:109:14\n            at [eval]-wrapper:6:24\n            at runScript (node:internal/process/execution:92:62)\n            at evalScript (node:internal/process/execution:123:10)\n            at node:internal/main/eval_string:51:3 {\n          generatedMessage: false,\n          code: 'ERR_ASSERTION',\n          actual: undefined,\n          expected: {\n            code: 'ERR_ACCESS_DENIED',\n            resource: 'd:\\\\workspace\\\\node-test-binary-windows-js-suites\\\\node\\\\test\\\\fixtures\\\\es-modules\\\\legacy-main-resolver\\\\index-js\\\\index'\n          },\n          operator: 'throws'\n        }\n        \n        Node.js v22.0.0-pre\n        \n      code: 'ERR_ASSERTION'\n      name: 'AssertionError'\n      expected: 0\n      actual: 1\n      operator: 'strictEqual'\n      stack: |-\n        [eval]:19:20\n        runScriptInThisContext (node:internal/vm:144:10)\n        node:internal/process/execution:109:14\n        [eval]-wrapper:6:24\n        runScript (node:internal/process/execution:92:62)\n        evalScript (node:internal/process/execution:123:10)\n        node:internal/main/eval_string:51:3 {\n        TestContext.<anonymous> (d:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\es-module\\test-cjs-legacyMainResolve-permission.js:75:14)\n        Test.runInAsyncScope (node:async_hooks:206:9)\n        Test.run (node:internal/test_runner/test:631:25)\n        Test.start (node:internal/test_runner/test:542:17)\n        node:internal/test_runner/test:946:71\n        node:internal/per_context/primordials:488:82\n        new Promise (<anonymous>)\n        new SafePromise (node:internal/per_context/primordials:456:29)\n        node:internal/per_context/primordials:488:9\n        Array.map (<anonymous>)\n      ...\n    # Subtest: should ensure permission model when resolving by packageJsonUrl\n    not ok 2 - should ensure permission model when resolving by packageJsonUrl\n      ---\n      duration_ms: 371.7335\n      location: 'd:\\\\workspace\\\\node-test-binary-windows-js-suites\\\\node\\\\test\\\\es-module\\\\test-cjs-legacyMainResolve-permission.js:79:3'\n      failureType: 'testCodeFailure'\n      error: |-\n        node:assert:126\n          throw new AssertionError(obj);\n          ^\n        \n        AssertionError [ERR_ASSERTION]: Missing expected exception.\n            at [eval]:20:20\n            at runScriptInThisContext (node:internal/vm:144:10)\n            at node:internal/process/execution:109:14\n            at [eval]-wrapper:6:24\n            at runScript (node:internal/process/execution:92:62)\n            at evalScript (node:internal/process/execution:123:10)\n            at node:internal/main/eval_string:51:3 {\n          generatedMessage: false,\n          code: 'ERR_ASSERTION',\n          actual: undefined,\n          expected: {\n            code: 'ERR_ACCESS_DENIED',\n            resource: 'd:\\\\workspace\\\\node-test-binary-windows-js-suites\\\\node\\\\test\\\\fixtures\\\\es-modules\\\\legacy-main-resolver\\\\index-json\\\\index.json'\n          },\n          operator: 'throws'\n        }\n        \n        Node.js v22.0.0-pre\n        \n      code: 'ERR_ASSERTION'\n      name: 'AssertionError'\n      expected: 0\n      actual: 1\n      operator: 'strictEqual'\n      stack: |-\n        [eval]:20:20\n        runScriptInThisContext (node:internal/vm:144:10)\n        node:internal/process/execution:109:14\n        [eval]-wrapper:6:24\n        runScript (node:internal/process/execution:92:62)\n        evalScript (node:internal/process/execution:123:10)\n        node:internal/main/eval_string:51:3 {\n        TestContext.<anonymous> (d:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\es-module\\test-cjs-legacyMainResolve-permission.js:133:14)\n        Test.runInAsyncScope (node:async_hooks:206:9)\n        Test.run (node:internal/test_runner/test:631:25)\n        Suite.processPendingSubtests (node:internal/test_runner/test:374:18)\n        Test.postRun (node:internal/test_runner/test:715:19)\n        Test.run (node:internal/test_runner/test:673:12)\n        process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n        async Promise.all (index 0)\n        async Suite.run (node:internal/test_runner/test:948:7)\n        async startSubtest (node:internal/test_runner/harness:216:3)\n      ...\n    1..2\nnot ok 1 - legacyMainResolve\n  ---\n  duration_ms: 752.5937\n  type: 'suite'\n  location: 'd:\\\\workspace\\\\node-test-binary-windows-js-suites\\\\node\\\\test\\\\es-module\\\\test-cjs-legacyMainResolve-permission.js:18:1'\n  failureType: 'subtestsFailed'\n  error: '2 subtests failed'\n  code: 'ERR_TEST_FAILURE'\n  ...\n1..1\n# tests 2\n# suites 1\n# pass 0\n# fail 2\n# cancelled 0\n# skipped 0\n# todo 0\n# duration_ms 774.879\n(node:6692) ExperimentalWarning: Permission is an experimental feature\n(Use `node --trace-warnings ...` to show where the warning was created)\n(node:6692) SecurityWarning: The flag --allow-child-process must be used with extreme caution. It could invalidate the permission model."},"time":1272.005,"fullname":"es-module/test-cjs-legacyMainResolve-permission","closingTestPoint":false}},{"testName":"parallel/test-cli-permission-deny-fs","timestamp":1701217871799,"status":"REGRESSION","jobName":"node-test-binary-windows-js-suites","config":"node-test-binary-windows-js-suites » 2,win11-arm64-COMPILED_BY-vs2022-arm64 #24735","buildUrl":"https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=2,nodes=win11-arm64-COMPILED_BY-vs2022-arm64/24735/","buildNumber":24735,"builtOn":"test-azure_msft-win11_vs2022-arm64-4","nodeVersion":"22.0.0","commitHash":"b2649a3bb95e6a322396f5c6e4a70a9e4a9c110e","callStack":[{"upstreamBuild":59529,"upstreamProject":"node-test-commit-windows-fanned","upstreamUrl":"job/node-test-commit-windows-fanned/"},{"upstreamBuild":66977,"upstreamProject":"node-test-commit","upstreamUrl":"job/node-test-commit/"},{"upstreamBuild":55992,"upstreamProject":"node-test-pull-request","upstreamUrl":"job/node-test-pull-request/"},{"prId":"50758"}],"tap":{"ok":false,"name":"parallel/test-cli-permission-deny-fs","id":140,"buffered":false,"tapError":null,"skip":false,"todo":false,"previous":null,"plan":null,"diag":{"severity":"fail","exitcode":1,"stack":"node:assert:1035\n    throw err;\n    ^\n\nAssertionError [ERR_ASSERTION]: The input did not match the regular expression /resource: '.*?[\\\\/](?:etc|passwd)'/. Input:\n\n'(node:2896) ExperimentalWarning: Permission is an experimental feature\\n' +\n  '(Use `node --trace-warnings ...` to show where the warning was created)\\n'\n\n    at Object.<anonymous> (d:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\parallel\\test-cli-permission-deny-fs.js:150:10)\n    at Module._compile (node:internal/modules/cjs/loader:1375:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1434:10)\n    at Module.load (node:internal/modules/cjs/loader:1206:32)\n    at Module._load (node:internal/modules/cjs/loader:1022:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:142:12)\n    at node:internal/main/run_main_module:28:49 {\n  generatedMessage: true,\n  code: 'ERR_ASSERTION',\n  actual: '(node:2896) ExperimentalWarning: Permission is an experimental feature\\n' +\n    '(Use `node --trace-warnings ...` to show where the warning was created)\\n',\n  expected: /resource: '.*?[\\\\/](?:etc|passwd)'/,\n  operator: 'match'\n}\n\nNode.js v22.0.0-pre"},"time":2774.005,"fullname":"parallel/test-cli-permission-deny-fs","closingTestPoint":false}},{"testName":"parallel/test-permission-fs-symlink","timestamp":1701217871799,"status":"REGRESSION","jobName":"node-test-binary-windows-js-suites","config":"node-test-binary-windows-js-suites » 2,win11-arm64-COMPILED_BY-vs2022-arm64 #24735","buildUrl":"https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=2,nodes=win11-arm64-COMPILED_BY-vs2022-arm64/24735/","buildNumber":24735,"builtOn":"test-azure_msft-win11_vs2022-arm64-4","nodeVersion":"22.0.0","commitHash":"b2649a3bb95e6a322396f5c6e4a70a9e4a9c110e","callStack":[{"upstreamBuild":59529,"upstreamProject":"node-test-commit-windows-fanned","upstreamUrl":"job/node-test-commit-windows-fanned/"},{"upstreamBuild":66977,"upstreamProject":"node-test-commit","upstreamUrl":"job/node-test-commit/"},{"upstreamBuild":55992,"upstreamProject":"node-test-pull-request","upstreamUrl":"job/node-test-pull-request/"},{"prId":"50758"}],"tap":{"ok":false,"name":"parallel/test-permission-fs-symlink","id":579,"buffered":false,"tapError":null,"skip":false,"todo":false,"previous":null,"plan":null,"diag":{"severity":"fail","exitcode":1,"stack":"node:assert:126\n  throw new AssertionError(obj);\n  ^\n\nAssertionError [ERR_ASSERTION]: node:assert:408\n\n    throw err;\n\n    ^\n\n\n\nAssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:\n\n\n\n  assert.ok(!process.permission.has('fs.read', blockedFile))\n\n\n\n    at Object.<anonymous> (d:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\fixtures\\permission\\fs-symlink.js:15:10)\n\n    at Module._compile (node:internal/modules/cjs/loader:1375:14)\n\n    at Module._extensions..js (node:internal/modules/cjs/loader:1434:10)\n\n    at Module.load (node:internal/modules/cjs/loader:1206:32)\n\n    at Module._load (node:internal/modules/cjs/loader:1022:12)\n\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:142:12)\n\n    at node:internal/main/run_main_module:28:49 {\n\n  generatedMessage: true,\n\n  code: 'ERR_ASSERTION',\n\n  actual: false,\n\n  expected: true,\n\n  operator: '=='\n\n}\n\n\n\nNode.js v22.0.0-pre\n\n\n    at Object.<anonymous> (d:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\parallel\\test-permission-fs-symlink.js:53:10)\n    at Module._compile (node:internal/modules/cjs/loader:1375:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1434:10)\n    at Module.load (node:internal/modules/cjs/loader:1206:32)\n    at Module._load (node:internal/modules/cjs/loader:1022:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:142:12)\n    at node:internal/main/run_main_module:28:49 {\n  generatedMessage: false,\n  code: 'ERR_ASSERTION',\n  actual: 1,\n  expected: 0,\n  operator: 'strictEqual'\n}\n\nNode.js v22.0.0-pre"},"time":738.005,"fullname":"parallel/test-permission-fs-symlink","closingTestPoint":false}},{"testName":"parallel/test-cli-permission-deny-fs","timestamp":1701217871799,"status":"REGRESSION","jobName":"node-test-binary-windows-js-suites","config":"node-test-binary-windows-js-suites » 3,win2019-COMPILED_BY-vs2022 #24735","buildUrl":"https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=3,nodes=win2019-COMPILED_BY-vs2022/24735/","buildNumber":24735,"builtOn":"test-rackspace-win2019_vs2019-x64-3","nodeVersion":"22.0.0","commitHash":"b2649a3bb95e6a322396f5c6e4a70a9e4a9c110e","callStack":[{"upstreamBuild":59529,"upstreamProject":"node-test-commit-windows-fanned","upstreamUrl":"job/node-test-commit-windows-fanned/"},{"upstreamBuild":66977,"upstreamProject":"node-test-commit","upstreamUrl":"job/node-test-commit/"},{"upstreamBuild":55992,"upstreamProject":"node-test-pull-request","upstreamUrl":"job/node-test-pull-request/"},{"prId":"50758"}],"tap":{"ok":false,"name":"parallel/test-cli-permission-deny-fs","id":139,"buffered":false,"tapError":null,"skip":false,"todo":false,"previous":null,"plan":null,"diag":{"severity":"fail","exitcode":1,"stack":"node:assert:1035\n    throw err;\n    ^\n\nAssertionError [ERR_ASSERTION]: The input did not match the regular expression /resource: '.*?[\\\\/](?:etc|passwd)'/. Input:\n\n'(node:3600) ExperimentalWarning: Permission is an experimental feature\\n' +\n  '(Use `node --trace-warnings ...` to show where the warning was created)\\n'\n\n    at Object.<anonymous> (C:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\parallel\\test-cli-permission-deny-fs.js:150:10)\n    at Module._compile (node:internal/modules/cjs/loader:1375:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1434:10)\n    at Module.load (node:internal/modules/cjs/loader:1206:32)\n    at Module._load (node:internal/modules/cjs/loader:1022:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:142:12)\n    at node:internal/main/run_main_module:28:49 {\n  generatedMessage: true,\n  code: 'ERR_ASSERTION',\n  actual: '(node:3600) ExperimentalWarning: Permission is an experimental feature\\n' +\n    '(Use `node --trace-warnings ...` to show where the warning was created)\\n',\n  expected: /resource: '.*?[\\\\/](?:etc|passwd)'/,\n  operator: 'match'\n}\n\nNode.js v22.0.0-pre"},"time":2203.015,"fullname":"parallel/test-cli-permission-deny-fs","closingTestPoint":false}},{"testName":"parallel/test-permission-fs-wildcard","timestamp":1701217871799,"status":"REGRESSION","jobName":"node-test-binary-windows-js-suites","config":"node-test-binary-windows-js-suites » 3,win2019-COMPILED_BY-vs2022 #24735","buildUrl":"https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=3,nodes=win2019-COMPILED_BY-vs2022/24735/","buildNumber":24735,"builtOn":"test-rackspace-win2019_vs2019-x64-3","nodeVersion":"22.0.0","commitHash":"b2649a3bb95e6a322396f5c6e4a70a9e4a9c110e","callStack":[{"upstreamBuild":59529,"upstreamProject":"node-test-commit-windows-fanned","upstreamUrl":"job/node-test-commit-windows-fanned/"},{"upstreamBuild":66977,"upstreamProject":"node-test-commit","upstreamUrl":"job/node-test-commit/"},{"upstreamBuild":55992,"upstreamProject":"node-test-pull-request","upstreamUrl":"job/node-test-pull-request/"},{"prId":"50758"}],"tap":{"ok":false,"name":"parallel/test-permission-fs-wildcard","id":577,"buffered":false,"tapError":null,"skip":false,"todo":false,"previous":null,"plan":null,"diag":{"severity":"fail","exitcode":1,"stack":"node:assert:126\n  throw new AssertionError(obj);\n  ^\n\nAssertionError [ERR_ASSERTION]: node:assert:408\n\n    throw err;\n\n    ^\n\n\n\nAssertionError [ERR_ASSERTION]: false == true\n\n    at [eval]:6:16\n\n    at runScriptInThisContext (node:internal/vm:144:10)\n\n    at node:internal/process/execution:109:14\n\n    at [eval]-wrapper:6:24\n\n    at runScript (node:internal/process/execution:92:62)\n\n    at evalScript (node:internal/process/execution:123:10)\n\n    at node:internal/main/eval_string:51:3 {\n\n  generatedMessage: true,\n\n  code: 'ERR_ASSERTION',\n\n  actual: false,\n\n  expected: true,\n\n  operator: '=='\n\n}\n\n\n\nNode.js v22.0.0-pre\n\n\n    at Object.<anonymous> (C:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\parallel\\test-permission-fs-wildcard.js:51:10)\n    at Module._compile (node:internal/modules/cjs/loader:1375:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1434:10)\n    at Module.load (node:internal/modules/cjs/loader:1206:32)\n    at Module._load (node:internal/modules/cjs/loader:1022:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:142:12)\n    at node:internal/main/run_main_module:28:49 {\n  generatedMessage: false,\n  code: 'ERR_ASSERTION',\n  actual: 1,\n  expected: 0,\n  operator: 'strictEqual'\n}\n\nNode.js v22.0.0-pre"},"time":384.983,"fullname":"parallel/test-permission-fs-wildcard","closingTestPoint":false}},{"testName":"parallel/test-permission-fs-write","timestamp":1701217871799,"status":"REGRESSION","jobName":"node-test-binary-windows-js-suites","config":"node-test-binary-windows-js-suites » 3,win2019-COMPILED_BY-vs2022 #24735","buildUrl":"https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=3,nodes=win2019-COMPILED_BY-vs2022/24735/","buildNumber":24735,"builtOn":"test-rackspace-win2019_vs2019-x64-3","nodeVersion":"22.0.0","commitHash":"b2649a3bb95e6a322396f5c6e4a70a9e4a9c110e","callStack":[{"upstreamBuild":59529,"upstreamProject":"node-test-commit-windows-fanned","upstreamUrl":"job/node-test-commit-windows-fanned/"},{"upstreamBuild":66977,"upstreamProject":"node-test-commit","upstreamUrl":"job/node-test-commit/"},{"upstreamBuild":55992,"upstreamProject":"node-test-pull-request","upstreamUrl":"job/node-test-pull-request/"},{"prId":"50758"}],"tap":{"ok":false,"name":"parallel/test-permission-fs-write","id":580,"buffered":false,"tapError":null,"skip":false,"todo":false,"previous":null,"plan":null,"diag":{"severity":"fail","exitcode":1,"stack":"node:assert:126\n  throw new AssertionError(obj);\n  ^\n\nAssertionError [ERR_ASSERTION]: node:assert:408\n\n    throw err;\n\n    ^\n\n\n\nAssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:\n\n\n\n  assert.ok(!process.permission.has('fs.write', blockedFolder))\n\n\n\n    at Object.<anonymous> (C:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\fixtures\\permission\\fs-write.js:20:10)\n\n    at Module._compile (node:internal/modules/cjs/loader:1375:14)\n\n    at Module._extensions..js (node:internal/modules/cjs/loader:1434:10)\n\n    at Module.load (node:internal/modules/cjs/loader:1206:32)\n\n    at Module._load (node:internal/modules/cjs/loader:1022:12)\n\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:142:12)\n\n    at node:internal/main/run_main_module:28:49 {\n\n  generatedMessage: true,\n\n  code: 'ERR_ASSERTION',\n\n  actual: false,\n\n  expected: true,\n\n  operator: '=='\n\n}\n\n\n\nNode.js v22.0.0-pre\n\n\n    at Object.<anonymous> (C:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\parallel\\test-permission-fs-write.js:44:10)\n    at Module._compile (node:internal/modules/cjs/loader:1375:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1434:10)\n    at Module.load (node:internal/modules/cjs/loader:1206:32)\n    at Module._load (node:internal/modules/cjs/loader:1022:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:142:12)\n    at node:internal/main/run_main_module:28:49 {\n  generatedMessage: false,\n  code: 'ERR_ASSERTION',\n  actual: 1,\n  expected: 0,\n  operator: 'strictEqual'\n}\n\nNode.js v22.0.0-pre"},"time":471.004,"fullname":"parallel/test-permission-fs-write","closingTestPoint":false}},{"testName":"parallel/test-permission-fs-symlink","timestamp":1701217871799,"status":"REGRESSION","jobName":"node-test-binary-windows-js-suites","config":"node-test-binary-windows-js-suites » 1,win2016-COMPILED_BY-vs2022-x86 #24735","buildUrl":"https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=1,nodes=win2016-COMPILED_BY-vs2022-x86/24735/","buildNumber":24735,"builtOn":"test-azure_msft-win2016_vs2017-x64-3","nodeVersion":"22.0.0","commitHash":"b2649a3bb95e6a322396f5c6e4a70a9e4a9c110e","callStack":[{"upstreamBuild":59529,"upstreamProject":"node-test-commit-windows-fanned","upstreamUrl":"job/node-test-commit-windows-fanned/"},{"upstreamBuild":66977,"upstreamProject":"node-test-commit","upstreamUrl":"job/node-test-commit/"},{"upstreamBuild":55992,"upstreamProject":"node-test-pull-request","upstreamUrl":"job/node-test-pull-request/"},{"prId":"50758"}],"tap":{"ok":false,"name":"parallel/test-permission-fs-symlink","id":580,"buffered":false,"tapError":null,"skip":false,"todo":false,"previous":null,"plan":null,"diag":{"severity":"fail","exitcode":1,"stack":"node:assert:126\n  throw new AssertionError(obj);\n  ^\n\nAssertionError [ERR_ASSERTION]: node:assert:408\n\n    throw err;\n\n    ^\n\n\n\nAssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:\n\n\n\n  assert.ok(!process.permission.has('fs.read', blockedFile))\n\n\n\n    at Object.<anonymous> (C:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\fixtures\\permission\\fs-symlink.js:15:10)\n\n    at Module._compile (node:internal/modules/cjs/loader:1375:14)\n\n    at Module._extensions..js (node:internal/modules/cjs/loader:1434:10)\n\n    at Module.load (node:internal/modules/cjs/loader:1206:32)\n\n    at Module._load (node:internal/modules/cjs/loader:1022:12)\n\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:142:12)\n\n    at node:internal/main/run_main_module:28:49 {\n\n  generatedMessage: true,\n\n  code: 'ERR_ASSERTION',\n\n  actual: false,\n\n  expected: true,\n\n  operator: '=='\n\n}\n\n\n\nNode.js v22.0.0-pre\n\n\n    at Object.<anonymous> (C:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\parallel\\test-permission-fs-symlink.js:53:10)\n    at Module._compile (node:internal/modules/cjs/loader:1375:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1434:10)\n    at Module.load (node:internal/modules/cjs/loader:1206:32)\n    at Module._load (node:internal/modules/cjs/loader:1022:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:142:12)\n    at node:internal/main/run_main_module:28:49 {\n  generatedMessage: false,\n  code: 'ERR_ASSERTION',\n  actual: 1,\n  expected: 0,\n  operator: 'strictEqual'\n}\n\nNode.js v22.0.0-pre"},"time":445.898,"fullname":"parallel/test-permission-fs-symlink","closingTestPoint":false}}]}